File is dat5.zip, the list of SNPs with very high Fst values.
You find chr1-10.out that are tab-delimited files.
Each line contains the information of each SNP.
SNPs are divided into "Genetic Unit" defined in our manuscript by "###".
As shown below, the list is too complicated and messy.
Tell me what kind of information you need.  I will extract the information that you need.


# Columns:
1st: chromosome
2nd: position on a chromosome ver. 2
3rd: First/Second alleles (allele 1 and 2 below)
4th: Freq. teosinte 1
5th: Freq. teosinte 2
6th: Freq. Mex low 1
7th: Freq. Mex low 2
8th: Freq. Mex high 1
9th: Freq. Mex high 2
10th: Freq. SA low 1
11th: Freq. SA low 2
12th: Freq. SA high 1
13th: Freq. SA high 2
14th: A1: Significant Fst values both in Mex and SA
        B1: Significant Fst values in Mex, polymorphic in SA
        B2: Significant Fst values in Mex, monomorphic in SA
        C1: Significant Fst values in SA, polymorphic in Mex
        C2: Significant Fst values in SA, monomorphic in Mex
15-18th: P-value of PHS
19th: my memo
20th: 1: PHS test is consistent with Fst test
        0: inconsistent
        2: complicated pattern
21th: describe something if teosinte and maize have three alleles, 
22th: Genes on which or near to which a SNP is located
23th: SNPs on exons
24th: SNPs on introns
25th: SNPs on UTRs
26th: SNPs on intergenic regions, <10kp from the nearest gene
27th: SNPs on or near miRNA (all are "NA")
