\section*{Discussion}

Our analysis of diversity and population structure in maize landraces from Mexico and S. America is consistent with earlier archaeological \cite[]{Piperno_2006_69,Perry_2006_16511492,Grobman_2012_22307642} and genetic \citep{vanHeerwaarden_2011_21189301} work suggesting an independent origin of S. American highland maize. 
We used our genetic data to fit a model of historical population size change, and find no evidence of a bottleneck in Mexico but results suggesting a strong bottleneck followed by expansion in the highlands of S. America. 
Surprisingly, our models showed no support for a maize domestication bottleneck, apparently contradicting earlier work using Sanger sequence from a smaller number of protein-coding genes \cite[]{Eyre-Walker_1998_9539756,Tenaillon_2004_15014173,Wright_2005_15919994}. 
While our SNP set is enriched for protein-coding genes, it does include a large number of noncoding variants.
Diversity differences between maize and teosinte are greatest in protein-coding regions \citep{Hufford_2012_22660546}, and like \citet{Hufford_2012_22660546} we find more similar patterns of rare variants in noncoding regions (Figure~\ref{JFD}) .
Together, these data suggest earlier models may have overestimated the strength of the domestication bottleneck.
A more detailed comparison with data from teosinte will be required  to validate these results, however, but they at least suggest that a reanalysis of the demography of maize domestication may be in order.

\paragraph{we see little convergence}
We used SNPs with significant $F_{ST}$ $P$-values were enriched in intergenic regions rather than protein coding regions (60.0\% vs. 47.9\%, Fisher's Exact Test $P < 10^{-7}$ for Mexico; 62.0\% vs. 47.8\%, FET $P<10^{-5}$ for S. America). 
Wallace paper, Tanja paper

\paragraph{theory comparison}
Based on our spatially explicit population genetic model, convergent evolution involving identical nucleotide changes is quite unlikely under either scenarios of independent mutation or transit of Central America by undirected (diffusive) sharing of seed. 
However, independent mutations could be expected in kilobase-sized targets, suggesting there might be signal for genes that share adaptive changes.
%clarify what kb target means
These conclusions could change if we drastically underestimated the rate of very-long-distance sharing of seed, e.g.\ if sharing across hundreds of kilometers was common at some point.


\label{low power}
However, it is known that the effective recombination rate of maize is very high \cite[]{Tenaillon_2001_11470895}.
Linkage disequilibrium rapidly decays and reaches plateau around 100 bp (Figure~\ref{supp:LD}).
The density of SNPs was roughly 1 SNP per 20 kb, so we could miss some of the adaptive variants.
We believe that our conclusion holds even if more dense SNP dataset can be used, but the future genotyping technology may solve this problem.
\cite[]{tiffin2014advances}.

\label{other taxa}
The rarity of convergent evolution in maize contrasts with data from humans \citep{Tennessen_2011_21698142} showing selection on the same genes in multiple pairs of tropical and temperate populations.  
However, in both maize and humans the majority of adaptive variants appear to have been derived from standing variation \cite[]{Tennessen_2011_21698142}.
One difference between these two species is effective population size: the effective population size of maize ($\sim10^5$) is an order of magnitude larger than that in humans \cite[]{Takahata_1997_9114074}.
Humans would therefore have less standing variation as a source of adaptation, perhaps resulting in the same variants being selected in multiple subpopulations (as long as $s$ is sufficiently large and initial frequency is, for example, $>0.1$).
In contrast, maize could maintain a larger number of adaptive variants in the ancestral lowland population.
In this case, if genetic variants produce similar phenotypic effects, they may be selected to high frequency in independent highland regions at random.
Or if Mexican and S. American highlands have slightly different climates, it is feasible that different variants are selected.
The target size of mutations can also increase the variants for adaptation, but there are currently no data regarding mutational target size in maize versus humans.
\plr{You are discussing adaptation from standing variation; what about from new mutation?  This wouldn't require postulating lots of equivalent variants?} \mbh{Sho's data seem to suggest predominance of adaptation from standing variation.  But, given that maize populations are larger than human populations, new mutations would also be more common in maize, which would, I think, lead to less convergence in maize than humans}


\label{other explanations}
\paragraph{standing genetic variation}
%These results suggest that maize adaptation to high elevation has largely made use of standing genetic variation. 
%Recent empirical examples of adaptation from standing variation \cite[Reviewed in ][]{Barrett_2008_18006185,Messer_2013_24075201} and detection of soft selective sweeps in Drosophila \cite[]{Garud_2013_ArXiv} and humans \cite[]{Turchin_2012_22902787,Peter_2012_23071458} suggest this may be a common form of adaptation. 
%In the case of maize, lowland-highland divergence events occurred very recently \cite[]{Piperno_2006_69,Perry_2006_16511492,Grobman_2012_22307642} and it would be unlikely that adaptation via \emph{de novo} mutations frequently occurred due to the waiting time required for new beneficial mutations to arise. \jri{tempted to drop this sentence as it seems to preempt our own analytical result? thoughts?}
%Theory also predicts adaptation from standing variation based on estimates of the demography of maize.
%Selection from standing variation should be common when the scaled mutation rate (the product of the effective population size, mutation rate and target size), $\Theta\geq1$, as long as the scaled selection coefficient (product of the effective population size and selection coefficient) $Ns$ is large enough \cite[]{Hermisson_2005_15716498}.
%Estimates of $\theta$ from synonymous nucleotide diversity in maize ($\cong0.014$ \cite[\emph{e.g.,} ][]{Tenaillon_2004_15014173,Wright_2005_15919994,Ross-Ibarra_2009_19153259}) then suggest adaptation from standing genetic variation would be likely for target sizes larger than a few hundred nucleotides. \jri{i think we say 0.018 earlier. Where does this 0.014 come from?} \st{0.018 is pi in \emph{parviglumis} that is used to estimate the size of the ancestral population,  0.014 is pi in maize.}
In fact, adaptation from multiple standing variants that produce similar phenotypes has previously been observed in maize: the \emph{grassy tillers1} (\emph{gt1}) gene \cite[]{Wills_2013_23825971} contains two artificially selected mutations that reduce ear number.
These mutations both segregate at low frequency in \emph{parviglumis} but have been individually selected to high frequency in different populations of maize.





