 \section*{Discussion}

Our analysis of diversity and population structure in maize landraces from Mesoamerica and S. America points to an independent origin of S. American highland maize, in line with earlier archaeological \cite[]{Piperno_2006_69,Perry_2006_16511492,Grobman_2012_22307642} and genetic \citep{vanHeerwaarden_2011_21189301} work. 
We use our genetic data to fit a model of historical population size change, and find 
%no evidence of a bottleneck in Mesoamerica but 
evidence of a strong bottleneck followed by expansion in the highlands of S. America. 
%\st{ {\bf Delete this para later} Surprisingly, our models showed no support for a maize domestication bottleneck, apparently contradicting earlier work \cite[]{Eyre-Walker_1998_9539756,Tenaillon_2004_15014173,Wright_2005_15919994}. 
%One factor contributing to these differences is the set of loci sampled. 
%Previous efforts focused on data exclusively from protein-coding regions, while our data set includes a large number of noncoding variants.
%Diversity differences between maize and teosinte are greatest in protein-coding regions \citep{Hufford_2012_22660546}, presumably due to the effects of background selection \citep{Charlesworth_1993_8375663}, and demographic estimates using only protein-coding loci should thus overestimate the strength of a domestication bottleneck.
%While a more detailed comparison with data from teosinte will be required to validate these results, they nonetheless suggest the value of a reassessment of the combined impacts of demography and selection on genome-wide patterns of diversity during maize domestication.}
We identified SNPs deviating from patterns of allele frequencies determined by our demographic model as loci putatively under selection for highland adaptation. 

\rev{Though the rapid decay of linkage disequilibrium in maize (Figure~\ref{supp:LD}) makes it likely we have identified only a subset of selected loci \cite[]{tiffin2014advances}, several lines of evidence suggest our results are likely representative of genome-wide patterns. SNPs identified as $F_{ST}$ outliers by our method show evidence of longer haplotypes and patterns of among-population allele frequency consistent with adaptation (Table~\ref{supp:phs}). 
Consistent with previous work suggesting adaptive introgression from teosinte, the Mesoamerican highland population shares a larger proportion of SNPs identified as adaptive in teosinte \citep{Pyhajarvi2013}.  We also see more $F_{ST}$ outliers Mesoamerica in regions introgressed from teosinte and which overlap with QTL for differences between \emph{parviglumis} and \emph{mexicana} \citep{Lauter_2004_15342532, Profford_2013}.  Finally, though our SNP data are enriched in low-copy genic regions, our results are consistent with both GWAS in maize  \citep{Wallace_2014_25474422} and local adaptation in teosinte \citep{Pyhajarvi2013} in finding an excess of putatively adaptive SNPs in intergenic regions of the genome.}

Although our data identify hundreds of loci that may have been targeted by natural selection in Mesoamerica and S. America, 
fewer than 1.8\% of SNPs and 2.1\% of genes show evidence for convergent evolution between the two highland populations.
This relative lack of convergent evolution is concordant with recently developed theory \citep{ralph2014convergent},
which applied to this system suggests that convergent evolution involving identical nucleotide changes 
is unlikely to have occurred in the time since highland colonization through either recurrent mutation or migration across Central America via seed sharing.   
These results are generally robust to variation in most of the parameters, but are sensitive to gross misestimation of some of the parameters -- for example if seed sharing was common over distances of hundreds of kilometers.  
The modeling highlights that our outlier approach may not detect traits undergoing convergent evolution 
if the genetic architecture of the trait is such that mutation at a large number of nucleotides would have equivalent effects on fitness 
(i.e. adaptive traits have a large mutational target). 
While QTL analysis suggests that some of the traits suggested to be adaptive in highland conditions may be determined by only a few loci \citep{Lauter_2004_15342532}, 
others such as flowering time \citep{buckler2009genetic} are likely to be the result of a large number of loci, each with small and perhaps similar effects on phenotype.  
Future quantitative genetic analysis of highland traits using genome-wide association methods may prove useful in searching for the signal of selection on such highly quantitative traits. 

Our observation of little convergent evolution is also consistent with the possibility that much of the adaptation to highland environments made use of standing genetic variation in lowland populations. 
Indeed, we find that as much as 90\% of the putatively adaptive variants in Mesoamerica and S. America are segregating in lowland populations, 
and the vast majority are also segregating in teosinte.  
Selection from standing variation should be common when the scaled mutation rate $\theta$ 
(product of the effective population size, mutation rate and target size) is greater than 1,
as long as the scaled selection coefficient $Ns$ 
(product of the effective population size and selection coefficient) is reasonably large \cite[]{Hermisson_2005_15716498}.
Estimates of $\theta$ from synonymous nucleotide diversity in maize  \citep{Tenaillon_2004_15014173,Wright_2005_15919994,Ross-Ibarra_2009_19153259}, 
suggest that adaptation from standing genetic variation may be likely for target sizes larger than a few hundred nucleotides.
In maize, such a scenario has been recently shown for the locus \emph{grassy tillers1} \cite[]{Wills_2013_23825971}, at which adaptive variants in both an upstream control region and the 3' UTR are segregating in teosinte but show evidence of recent selection in maize, presumably due to the effects of this locus on branching and ear number.

\rev{Both our empirical and theoretical results suggest that adaptation to high elevation probably occurred through some combination of selection on standing variation and independent \textit{de novo} mutation at highly quantitative traits. 
Because cultivated maize has retained high levels of diversity,
much of the ancestral variation present in the populations that founded each of the two highlands was likely shared,
allowing for the possibility of shared signals due to selection on the same ancestral variants.
However, initial frequencies of alleles present as standing variation will be highly stochastic,
leading to a significant role of chance in which alleles are selected,
as well the strength of the signal of $F_{ST}$.
This is particularly true for alleles likely to be adaptive in the highlands and thus weakly deleterious in lowland populations, as these should be rare in individual populations. 
Epistasis could make it even less likely that the same allele is shared between regions. Overall, our results highlight the complexity of studying convergent evolution for quantitative traits in highly diverse species. Our future efforts will take advantage of reciprocal transplant experiments to identify specific phenotypes under selection. We are also developing mapping populations in both Mesoamerica and S. America that should allow identification of genomic regions underlying phenotypes of interest and estimation of the proportion of adaptive variation shared between populations.  }

%\label{other taxa}
%The rarity of convergent evolution in maize contrasts with data from humans \citep{Tennessen_2011_21698142} showing selection on the same genes in multiple pairs of tropical and temperate populations.  
%However, in both maize and humans the majority of adaptive variants appear to have been derived from standing variation \cite[]{Tennessen_2011_21698142}.
%One difference between these two species is effective population size: the effective population size of maize ($\sim10^5$) is an order of magnitude larger than that in humans \cite[]{Takahata_1997_9114074}.
%Humans would therefore have less standing variation as a source of adaptation, perhaps resulting in the same variants being selected in multiple subpopulations (as long as $s$ is sufficiently large and initial frequency is, for example, $>0.1$).
%In contrast, maize could maintain a larger number of adaptive variants in the ancestral lowland population.
%In this case, if genetic variants produce similar phenotypic effects, they may be selected to high frequency in independent highland regions at random.
%Or if Mexican and S. American highlands have slightly different climates, it is feasible that different variants are selected.






%Recent empirical examples of adaptation from standing variation \cite[Reviewed in ][]{Barrett_2008_18006185,Messer_2013_24075201} and detection of soft selective sweeps in Drosophila \cite[]{Garud_2013_ArXiv} and humans \cite[]{Turchin_2012_22902787,Peter_2012_23071458} suggest this may be a common form of adaptation.




