\section*{Discussion}

Our analysis of diversity and population structure in maize landraces from Mexico and S. America points to an  an independent origin of S. American highland maize, in line with earlier archaeological \cite[]{Piperno_2006_69,Perry_2006_16511492,Grobman_2012_22307642} and genetic \citep{vanHeerwaarden_2011_21189301} work. 
We use our genetic data to fit a model of historical population size change, and find no evidence of a bottleneck in Mexico but a strong bottleneck followed by expansion in the highlands of S. America. 
Surprisingly, our models showed no support for a maize domestication bottleneck, apparently contradicting earlier work \cite[]{Eyre-Walker_1998_9539756,Tenaillon_2004_15014173,Wright_2005_15919994}. 
One factor contributing to these differences is the set of loci sampled. 
Previous efforts focused on data exclusively from protein-coding regions, while our data set includes a large number of noncoding variants.
Diversity differences between maize and teosinte are greatest in protein-coding regions \citep{Hufford_2012_22660546}, presumably due to the effects of background selection \citep{}, and demographic estimates using only protein-coding loci should thus overestimates the strength of a domestication bottleneck.
While a more detailed comparison with data from teosinte will be required to validate these results, they nonetheless suggest the value of a reassessment of the combined impacts of demography and selection on genome-wide patterns of diversity during maize domestication.

\paragraph{Highland Adaptation}
We identified SNPs deviating from patterns of allele frequencies determined by our demographic model as loci putatively under selection for highland adaptation.
These conclusions are supported by evidence of haplotype differentiation \jri{supp.} and the directionality of allele frequency change \jri{supp.}.
Consistent with results from both GWAS \citep{Wallace} and local adaptation in teosinte \citep{Tanja}, we find that putatively adaptive SNPs are enriched in intergenic regions of the genome, further suggesting an important role for regulatory variation in maize evolution. 

Although our data suggest that hundreds of loci were targeted by natural selection in Mexico and South America, fewer than $X\%$ of SNPs and $X\%$ of genes show evidence for convergent evolution between the two highland populations.
To evaluate the significance of our empirical observations given the biology and colonization history of highland maize, we applied recently developed models of convergent evolution \citep{Peter} across a range of parameter possibilities. \jri{is this fair to say? we didn't really do any formal sensitivity analyses...} 
Our modeling results suggest that convergent evolution involving identical nucleotide changes is quite unlikely due to recurrent mutation or diffusion across Central America via seed sharing.    



 these results are robust to a range of parameter values \jri{fair to say?}, independent mif many adaptive traits large number of nucleotides independent mutations could be expected in kilobase-sized targets, suggesting there might be signal for genes that share adaptive changes.
%clarify what kb target means
These conclusions could change if we drastically underestimated the rate of very-long-distance sharing of seed, e.g.\ if sharing across hundreds of kilometers was common at some point.

\label{other taxa}
The rarity of convergent evolution in maize contrasts with data from humans \citep{Tennessen_2011_21698142} showing selection on the same genes in multiple pairs of tropical and temperate populations.  
However, in both maize and humans the majority of adaptive variants appear to have been derived from standing variation \cite[]{Tennessen_2011_21698142}.
One difference between these two species is effective population size: the effective population size of maize ($\sim10^5$) is an order of magnitude larger than that in humans \cite[]{Takahata_1997_9114074}.
Humans would therefore have less standing variation as a source of adaptation, perhaps resulting in the same variants being selected in multiple subpopulations (as long as $s$ is sufficiently large and initial frequency is, for example, $>0.1$).
In contrast, maize could maintain a larger number of adaptive variants in the ancestral lowland population.
In this case, if genetic variants produce similar phenotypic effects, they may be selected to high frequency in independent highland regions at random.
Or if Mexican and S. American highlands have slightly different climates, it is feasible that different variants are selected.
The target size of mutations can also increase the variants for adaptation, but there are currently no data regarding mutational target size in maize versus humans.
\plr{You are discussing adaptation from standing variation; what about from new mutation?  This wouldn't require postulating lots of equivalent variants?} \mbh{Sho's data seem to suggest predominance of adaptation from standing variation.  But, given that maize populations are larger than human populations, new mutations would also be more common in maize, which would, I think, lead to less convergence in maize than humans}


\label{other explanations}
\paragraph{standing genetic variation}
%These results suggest that maize adaptation to high elevation has largely made use of standing genetic variation. 
%Recent empirical examples of adaptation from standing variation \cite[Reviewed in ][]{Barrett_2008_18006185,Messer_2013_24075201} and detection of soft selective sweeps in Drosophila \cite[]{Garud_2013_ArXiv} and humans \cite[]{Turchin_2012_22902787,Peter_2012_23071458} suggest this may be a common form of adaptation. 
%In the case of maize, lowland-highland divergence events occurred very recently \cite[]{Piperno_2006_69,Perry_2006_16511492,Grobman_2012_22307642} and it would be unlikely that adaptation via \emph{de novo} mutations frequently occurred due to the waiting time required for new beneficial mutations to arise. \jri{tempted to drop this sentence as it seems to preempt our own analytical result? thoughts?}
%Theory also predicts adaptation from standing variation based on estimates of the demography of maize.
%Selection from standing variation should be common when the scaled mutation rate (the product of the effective population size, mutation rate and target size), $\Theta\geq1$, as long as the scaled selection coefficient (product of the effective population size and selection coefficient) $Ns$ is large enough \cite[]{Hermisson_2005_15716498}.
%Estimates of $\theta$ from synonymous nucleotide diversity in maize ($\cong0.014$ \cite[\emph{e.g.,} ][]{Tenaillon_2004_15014173,Wright_2005_15919994,Ross-Ibarra_2009_19153259}) then suggest adaptation from standing genetic variation would be likely for target sizes larger than a few hundred nucleotides. \jri{i think we say 0.018 earlier. Where does this 0.014 come from?} \st{0.018 is pi in \emph{parviglumis} that is used to estimate the size of the ancestral population,  0.014 is pi in maize.}
In fact, adaptation from multiple standing variants that produce similar phenotypes has previously been observed in maize: the \emph{grassy tillers1} (\emph{gt1}) gene \cite[]{Wills_2013_23825971} contains two artificially selected mutations that reduce ear number.
These mutations both segregate at low frequency in \emph{parviglumis} but have been individually selected to high frequency in different populations of maize.

Finally, although we evaluated a genome-wide sample of more than 90,000 SNPs, this sampling is likely insufficient to capture all of the signals of selection across the genome.  
Linkage disequilibrium in mays decays rapidly \citep{Chia}, reaching a plateau in only a few hundred bp (Figure~\ref{supp:LD}) and a much greater density of SNPs would be needed to effectively identify the majority of selective sweeps in the history of these populations \cite[]{tiffin2014advances}.
SNP density alone does not explain the lack of convergent evolution seen at SNPs showing evidence of selection, however.  
Our genomic sampling may have thus identified only a subset of all loci targeted by natural selection, but there is no reason to believe that the percentage of selected loci showing convergent selection should change with higher genotyping density. 





