
%\newpage



\newpage

\section*{Appendix} %\label{apx:demographic_model}

\section*{Demographic modeling}

Throughout we use in many ways the {\em branching process approximation} -- if an allele is locally rare, then for at least a few generations, the fates of each offspring are nearly independent.
So, if the allele is locally deleterious, the total numbers of that allele behave as a subcritical  branching process, destined for ultimate extinction.
On the other hand, if the allele is advantageous, it will either die out or become locally common, with its fate determined in the first few generations.
If the number of offspring of an individual with this allele is the random variable $X$, 
with mean $\E[X] = 1+s$ (selective advantage $s>0$), variance $\var[X]=\xi^2$, and $\P\{X=0\}>0$ (some chance of leaving no offspring),
then the probability of local nonextinction $p_*$ is approximately $p_* \approx 2s/\xi^2$ to a second order in $s$.
The precise value can be found by defining the generating function $\Phi(u) = \E[u^X]$; 
the probability of local nonextinction $p_*$ is the minimal solution to $\Phi(1-u) = 1-u$.
(This can be seen because: $1-p_*$ is the probability that an individual's family dies out;
this is equal to the probability that the families of all that individuals' children die out;
since each child's family behaves independently, if the individual has $x$ offspring, this is equal to $(1-p_*)^x$;
and $\Phi(1-p_*) = \E[(1-p_*)^X]$.)

If the selective advantage ($s$) depends on geographic location, 
a similar fact holds: index spatial location by $i \in 1, \ldots, n$,
and for $u = (u_1,u_2,\ldots,u_n)$ define the functions $\Phi_i(u) = \E[ \prod_j u_j^{X_{ij}} ]$,
where $X_{ij}$ is the (random) number of offspring that an individual at $i$ produces at location $j$.
Then $p_* = (p_{*1}, \ldots, p_{*n})$, the vector of probabilities that a new mutation at each location eventually fixes,
is the minimal solution to $\Phi(1-p_*) = 1-p_*$,
i.e.\ $\Phi_i(1-p_*) = 1-p_{*i}$.

%%%%% in rcode/maize_calcs.R
% # maize
% N <- 103359  # plants per field
% Nf <- 561    # ears to make next generation
% e <- 0.068   # prob of total seed stock replacement
% mg <- .0083  # pollen migration
% pm <- .02    # prob of partial stock addition
% m <- 0.2     # proportion of stock replaced
% eqvals <- c(1, iterroot( function (p) { 
%             p.dead <- e + (1/2)*(1-e)*(1-Nf/N)
%             return( p.dead * 1 + 
%                     (1/2)*(1-e)*exp(((1+sb)/p.dead)*( p - 1 )) + 
%                     ((1/2)*(1-e)*Nf/N)*exp(((N/Nf)*(1+sb)/(1-e))*( p - 1 )) 
%                 )
%         } ) )
% genfn <- function (f,selx,migr=.5) { 
%         (e + (1/2)*(1-e)*(1-Nf/N))* 1 + #  no offspring
%         (1/2)*(1-e)*exp(((1+selx)/(1-e))*( f + migr*diff(c(eqvals[1],f,eqvals[2]),differences=2) - 1 )) + # pollen, Poisson mean 1+s(x)
%         (1/2)*(1-e)*(Nf/N)*exp(((N/Nf)*(1+selx/(1-e)))*( f + migr*diff(c(eqvals[1],f,eqvals[2]),differences=2) - 1 )) # seed, Poisson mean (1+s(x))*N/Nf
%     }

Here we consider a linear habitat, so that the selection coefficient at location $\ell_i$ is $s_i = \min( s_b, \max( - s_d, \alpha \ell_i ) )$.
There does not seem to be a nice analytic expression for $p_*$ in this case,
but since $1-p_*$ is a fixed point of $\Phi$, the solution can be found by iteration:
$1-p_* = \lim_{n \to \infty} \Phi^n(u)$ for an appropriate starting point $u$.

\subsection*{Maize model}

The migration and reproduction dynamics we use are taken largely from \citet{vanHeerwaarden2010}.  
On a large scale,
fields of $N$ plants are replanted each year from $N_f$ ears,
either from completely new stock (with probability $p_e$),
from partially new stock (a proportion $r_m$ with probability $p_m$),
or entirely from the same field.
Plants have an average of $\mu_E$ ears per plant, and ears have an average of $N/N_f$ kernels; 
so a plant has on average $\mu_E N/N_f$ kernels, and a field has on average $\mu_E N$ ears and $\mu_E N^2/N_f$ kernels. \jri{what happens if we change mean ears per plant from 3 to 1?}
We suppose that a plant with the selected allele is pollen parent to $(1+s) \mu_E N/N_f$ kernels,
and also seed parent to $(1+s)\mu_E N/N_f$ kernels, still in $\mu_E$ ears.
The number of offspring a plant has depends on how many of its offspring kernels get replanted.
Some proportion $m_g$ of the pollen-parent kernels are in other fields, and may be replanted;
but with probability $p_e$ no other kernels (i.e.~those in the same field) are replanted.
Otherwise, with probability $1-p_m$ the farmer chooses $N_f$ of the ears from this field to replant
(or, $(1-r_m) N_f$ of them, with probability $p_m$);
this results in a mean number $N_f/N$ (or, $(1-r_m)N_f/N$) of the plant's ears of seed children being chosen,
and a mean number $1+s$ of the plant's pollen children kernels being chosen.
Furthermore, the field is used to completely (or partially) replant another's field with chance $p_e/(1-p_e)$ (or $p_m$);
resulting in another $N_f/N$ (or $r_m N_f/N$) ears and $1+s$ (or $r_m (1+s)$) pollen children being replanted elsewhere.
Here we have assumed that pollen is well-mixed within a field,
and that the selected allele is locally rare.
Finally, we must divide all these offspring numbers by 2,
since we look at the offspring carrying a particular haplotype, not of the diploid plant's genome.

The above gives mean values; to get a probability model we assume that every count is Poisson.
In other words, we suppose that the number of pollen children is Poisson with random mean $\lambda_P$,
and the number of seed children is a mixture of $K$ independent Poissons with mean $(1+s)N/N_f$ each,
where $K$ is the random number of ears chosen to replant, which is itself Poisson with mean $\mu_K$.
By Poisson additivity, the numbers of local and migrant offspring are Poisson,
with means $\lambda_P = \lambda_{PL} + \lambda_{PM}$ and $\mu_K = \mu_{KL} + \mu_{KM}$ respectively.
With probability $p_e$, $\lambda_{PM} = m_g(1+s)$ and $\mu_K = \lambda_{PL} = 0$.
Otherwise, with probability $(1-p_e)(1-p_m)$, $\mu_{KL} = N_f/N$ and $\lambda_{PL} = (1+s)(1-m_g)$;
and with probability $(1-p_e)p_m$, $\mu_{KL} = (1-r_m) N_f/N$ and $\lambda_{PL} = (1-r_m)(1+s)(1-m_g)$.
The migrant means are,
with probability $(1-p_e) p_e/(1-p_e) = p_e$, $\mu_{KM} = N_f/N$ and $\lambda_{PM} = 1+s$;
while with probability $(1-p_e) p_m$, $\mu_{KM} = r_m N_f/N$ and $\lambda_{PM} = (1+s)(r_m(1-m_g) + m_g)$,
and otherwise $\mu_{KM} = 0$ and $\lambda_{PM} = m_g(1+s)$.


\begin{table}[htb!!]
  \begin{center}
  \begin{tabular}{|cll|}
    \hline
    complete seed stock replacement prob & $p_e$     & 0.068 \\
    pollen migration rate                & $m_g$      & 0.0083 \\
    number of plants per field           & $N$        & $10^5$ \\
    number of ears used to replant       & $N_f$      & 561 \\
    mean ears per plant                  & $\mu_E$    & 3 \\
    partial stock replacement prob       & $p_m$      & 0.02 \\
    mean proportion stock replaced       & $r_m$      & 0.2 \\
    pollen migration distance            & $\sigma_p$ & 0 km \\
    seed replacement distance            & $\sigma_s$ & 50 km \\
    distance between demes               & $a$        & 15 km \\
    width of altitudinal cline           & $w$        & 62km \\
    deleterious selection coefficient    & $s_d$      & varies \\
    beneficial selection coefficient     & $s_b$      & varies \\
    slope of selection gradient          & $\alpha$   & $(s_d+s_b)/w$ \\
    variance in offspring number         & $\xi^2$    & varies \\
    maize population density             & $\rho$     & $5 \times 10^3$ \\
    area of highland habitat             & $A$        & 500 km$^2$ \\
    mean dispersal distance              & $\sigma$   & 1.8 km \\
    \hline 
  \end{tabular}
\end{center}
  \caption{Parameter estimates used in calculations, and other notation.
  \label{tab:parameters}
  }
\end{table}


\subsection*{Math}

The generating function of a Poisson with mean $\lambda$ is $\phi(u;\lambda)=\exp(\lambda(u-1))$,
and the generating function of a Poisson($\mu$) sum of Poisson($\lambda$) values is $\phi(\phi(u;\lambda);\mu)$.
Therefore, the generating function for the diploid process, ignoring spatial structure,
is
\begin{align}
 \Phi(u) &= \begin{aligned}[t]
   & p_e \phi(u;m_g(1+s)) \\
   & \quad {} + \left\{ (1-p_e) (1-p_m) \phi(u;(1+s)(1-m_g))\phi(\phi(u;(1+s)N/N_f);N_f/N) \right. \\
   & \quad \qquad \left. {} + (1-p_e) p_m \phi(u;(1+s)(1-r_m)(1-m_g)) \phi(\phi(u;(1+s)N/N_f);(1-r_m)N_f/N) \right\} \\
   & \quad \quad {} \times \left\{ p_e/(1-p_e) \phi(u;1+s) \phi(\phi(u;(1+s)N_f/N);N_f/N) \right. \\
   & \quad \qquad \left. {} + p_m \phi(u;(1+s)(r_m(1-p_e)(1-m_g)+m_g)) \right. \\ 
   & \quad \qquad \quad \left. {}\times \phi(\phi(u;(1+s)N/N_f);r_m N_f/N)\right. \\
   & \quad \qquad \left. {} + (1 - p_e/(1-p_e)-p_m) \phi(u;m_g(1+s)) \right\}
 \end{aligned} \label{eqn:genfn} \\
 &= \begin{aligned}[t]
   & \phi(u;m_g(1+s)) \big( \; p_e \big.\\
   & \quad {} + \left\{ (1-p_e) (1-p_m) \phi(u;(1+s)(1-m_g))\phi(\phi(u;(1+s)N/N_f);N_f/N) \right. \\
   & \quad \qquad \left. {} + (1-p_e) p_m \phi(u;(1+s)(1-r_m)(1-m_g)) \phi(\phi(u;(1+s)N/N_f);(1-r_m)N_f/N) \right\} \\
   & \quad \quad {} \times \left\{ p_e/(1-p_e) \phi(u;(1+s)(1-m_g)) \phi(\phi(u;(1+s)N_f/N);N_f/N) \right. \\
   & \quad \qquad {} + p_m \phi(u;(1+s)r_m(1-m_g)) \\
   & \quad \qquad \quad {} \times \phi(\phi(u;(1+s)N/N_f);r_m N_f/N) \\
   & \quad \qquad \left. {} + (1-p_e/(1-p_e)-p_m) \right\}  \big)
 \end{aligned}
\end{align}
To get the generating function for a haploid, replace every instance of $1+s$ by $(1+s)/2$.

As a quick check,
% \begin{align}
%   \Phi(0) &= p_e + \left\{ (1-p_e)(1-p_m)+(1-p_e)p_m \right\}
%     \times \left\{ p_e/(1-p_e) + p_m + (1-p_e/(1-p_e)-p_m) \right\}
%   &= p_e + (1-p_e)
%   &= 1 .
% \end{align}
% And, 
the mean total number of offspring of a diploid is
\begin{align}
  & \begin{aligned}
(1+s) \big(
    m_g
    + & (1-p_e)\left\{ (1-p_m) ( (1-m_g) + 1 ) + p_m ( (1-r_m)(1-m_g) + (1-r_m) ) \right\} \\
    & \qquad + \left\{ p_e ( (1-m_g) + 1 ) + p_m (1-p_e)( r_m(1-m_g) + r_m ) \right\} 
    \big) 
  \end{aligned}  \\
& \qquad = 
    (1+s) \big( m_g + 
      (1-p_e)(2-m_g)(1 - p_m r_m)
      + ( p_e (2-m_g) + p_m r_m (1-p_e)(2-m_g) )
    \big) \\
& \qquad = (1+s) \big( 
    m_g + (2-m_g) ( (1-p_e) (1 - p_m r_m) + p_e + p_m r_m(1-p_e) )
  \big) \\
& \qquad = (1+s) \big( 
    m_g + (2-m_g)
  \big) \\
& \qquad = 2(1+s) .
\end{align}
\plr{CHECK}

We show numerically later that the probability of establishment is very close to $2s$ over the variance in reproductive number (as expected).
It is possible to write down an expression for the variance, but it's a big, ugly one that doesn't lend itself to intuition.

\subsection*{Migration and spatial structure}

To incorporate spatial structure, suppose that the locations $\ell_k$ are arranged in a regular grid, so that $\ell_k = a k$. 
Recall that $s_k$ is the selection coefficient at location $k$.
If the total number of offspring produced by an individual at $\ell_i$ is Poisson($\lambda_i$), with each offspring independently migrating to location $j$
with probability $m_{ij}$,
then the number of offspring at $j$ is Poisson($m_{ij}\lambda_i$),
and so the generating function is
\begin{align}
  \phi(u;\lambda,m) &= \prod_j \exp( \lambda_i m_{ij} ( u_j - 1 ) ) \\
  &= \exp\left\{ \lambda_i \left(\left(\sum_j m_{ij} u_j\right) - 1\right) \right\} .
\end{align}
We can then substitute this expression into equation \eqref{eqn:genfn},
with appropriate migration kernels for pollen and seed dispersal.

For migration, we need migration rates and migration distances for both wind-blown pollen
and for farmer seed exchange.
The rates are parameterized as above;
we need the typical dispersal distances, however.
One option is to say that the typical distance between villages is $d_v$,
and that villages are discrete demes,
so that pollen stays within the deme (pollen migration distance 0)
and seed is exchanged with others from nearby villages;
on average $\sigma_s$ distance away in a random direction.
The number of villages away the seed comes from could be geometric 
(including the possibility of coming from the same village).

The dispersal distance
-- the mean distance between parent and offspring --
is the average of the pollen and seed mean dispersal distances.
With the above assumptions, the pollen dispersal distance is zero,
and the seed dispersal distance is the chance of inter-village movement
multiplied by the mean distance moved.
This is
% pe <- .068; pm <- .02; rm <- .2; sigma.s <- 50
% (1/2) * (pe + (1-pe)*pm*rm) * sigma.s
\begin{align}
  \sigma = \frac{1}{2} (p_e + (1-p_e) p_m r_m ) \sigma_s = 1.7932 \text{km}
\end{align}
at the parameter values above.

Iterating the generating function above finds the probability of establishment as a function of distance along the cline.
This is shown in figure \ref{sfig:prob_estab}.
Note that the approximation $2s$ divided by the variance in offspring number is quite close.

\begin{figure}[ht!!]
  \begin{center}
    \includegraphics{prob-estab-62}
    \includegraphics{prob-estab-500}
  \end{center}
  \caption{
  \plr{make this look better}
  Probability of establishment, as a function of distance along and around an altitudinal cline, whose boundaries are marked by the green lines.
  {\bf (A)} The parameters above; with cline width 62km; {\bf (B)} the same, except with cline width 500km.
  \label{sfig:prob_estab}
  }
\end{figure}


As we show in \citet{ralph2014convergent}, 
the rate of adaptation by diffusive migration is roughly
\begin{align*}
  \migrate = \rho \frac{s_b \sqrt{2 s_m}}{2\xi^2} \exp\left(- \frac{\sqrt{2 s_m}R}{\sigma} \right) .
\end{align*}

\jri{do we need to expland? not sure. }
%But for now, let's interpret.

First note that for $10^{-1} \le s_m \le 10^{-4}$, the value $1/\sqrt{2s_m}$ is between 2 and 70 --
so the exponential decay of the chance of migration falls off on a scale of between 2 and 70 times the dispersal distance.
Above we have estimated the dispersal distance to be $\sigma \approx 2$ km,
and far below the mean distance $\sigma_s$ to the field that a farmer replants seed from, when this happens,
which we have as $\sigma_s = 50$ km.
Taking $\sigma=2$ km, we have that $4 \le \sigma/\sqrt{2s_m} \le 150$ km.
A very conservative upper bound might be $\sigma \le \sigma_s/20$ (if farmers replaced 10\% of their seed with long-distance seed every year).
At this upper bound, we would have $5 \le \sigma/\sqrt{2s_m} \le 175$ km,
which is not very different.
This makes the exponential term very small since $R$ is on the order of 1,000 km.

Taking $\sigma=2$ km, we then compute that 
% rho <- 5000; sb <- .01; sm <- 10^(-(1:4)); xisq <- 50; sigma <- 2
% sapply( 1000*(1:4), function (R) rho * sb * sqrt(2*sm) / (2 *  xisq) * exp(- sqrt(2*sm)*R/sigma ) )
if $s_m = 10^{-4}$ (very weak selection in the lowlands), then for $R=1,000$ km, the migration rate is $\migrate \le 10^{-5}$,
i.e.\ it would take on the order of 100,000 generations (years) to get a successful migrant only 1,000 km away,
under this model of undirected, diffusive dispersal.
For larger $s_m$, the migration rate is much smaller.

%below moved from methods.`
If highland alleles are neutral in the lowlands the situation is more difficult to model, but we can make some informed guesses.
For maize in the Andean highlands to have inherited a highland-adapted allele from the Mexican highlands, those Andean plants must be directly descended from highland Mexican plants that lived more recently than the appearance of the adaptive allele.
In other words, the ancestral lineages along which the modern Andean plants have inherited at that locus must trace back to the Mexican highlands.
If the allele is neutral in the lowlands, we can treat the movement of these lineages as a neutral process, using the framework of coalescent theory \citep{wakeley2005coalescent}.
To do this, we need to follow \emph{all} of the $N \approx 2.5 \times 10^6$ lineages backwards; these quickly coalesce to fewer $m$ lineages in approximately $\sum_{k=m}^N \frac{2N}{\xi^2 k(k+1)} \approx 1.25 \times 10^5/m$ generations, leaving about 1000 lineages after 100 generations that are spread over a larger area.
The displacement of a lineage after $m$ generations has variance $m \sigma^2$ and is approximately Gaussian.
If we assume that $n$ lineages are independent, and $Z_n$ is the distance to the furthest lineage, then $\P\{ Z_n / \sqrt{m \sigma^2} \le x /\sqrt{2 \log n} + \sqrt{2 \log n} - (1/2) (\log \log n + \log 4 \pi)/\sqrt{2 \log n} \} \approx \exp( - e^{-x} )$ \citep{berman1964limit}. \jri{Peter are the braces in the right place? seems to me naively it should be  $\P\{ Z_n / \sqrt{m \sigma^2} \le x \} = \sqrt{2 \log n} + ...$ ??}

%%%%%%%
\section{Adaptation by mutation}

\plr{just a placeholder for now; to be merged in}

First, we'd like to compute how difficult is it for the beneficial adaptation to arise by new mutation.
The rate of appearance of mutant alleles is a Poisson process,
and we can assume that each is successful or not independently,
so the time until the new mutant appears and fixes is exponentially distributed,
with rate equal to the mutation rate multiplied by the probability of establishment integrated over the population.
Referring to figure~\ref{sfig:prob_estab},
we see that this is pretty close to ( (area of high altitude) $+$ ($1/2$ area of altitudinal gradient) ) $\times$ (population density) $\times$ (prob of establishment at high altitude).

Let $A$ denote (area of high altitude) plus ($1/2$ area of altitudinal gradient).
The population density $\rho$ is roughly 0.5--5 people per km$^2 \times$ (0.5 ha field/person) $\times$ ($2\times10^4$ plants per field ha) $=$ (5000--50000 plants per km$^2$).
As a check, the other set of numbers was ``one village per 15 km''; i.e.\ per square with 15km on a side, which is 0.444 people per km$^2$.

Since the probability of establishment at high altitude is approximately $2 s_b / \xi^2$, 
with $\xi^2$ the variance in offspring number,
the rate of appearance is just 
\begin{align*}
  \mutrate = 2 \rho A s_b \mu / \xi^2 .
\end{align*}
% rho <- 5000; sb <- 10^(-(1:3)); xisq <- 50; A <- 500
% sapply( c(1e-8,1e-5), function (mu) 2 * rho * A * sb *mu / xisq )
At the values above, with $.1 \le s_b \le .001$, the factor $2 \rho A s_b / \xi^2$ multiplying the mutation rate
varies between $10^2$ and $10^5$,
implying that a single-base mutation with $\mu=10^{-8}$ would have to wait between $10^4$ and $10^6$ generations to fix,
but a mutation with a larger target, say $\mu=10^{-5}$, would fix in tens to thousands of generations, depending on the selection coefficient.



%%%%%%%
\section{Adaptation by migration}

%%%%%%%%%%%%
\section{Conclusion}
It seems unlikely that any alleles that are adaptive in the highlands and deleterious at all in the lowlands
would have transited central America by undirected (diffusive) sharing of seed.
The conclusions could change if we drastically underestimate the rate of very long distance sharing of seed,
e.g. if sharing across hundreds of kilometers was common at some point.

Both calculations are very pessimistic about the chance of shared single-base changes through either migration or independent mutation.
However, independent mutations could be expected in kilobase-size targets,
suggesting there might be signal for genes that share adaptive changes.


\newpage
\suppl
\section*{Supporting information}

\newpage

%\clearpage

% space of double hline in Table
\doublerulesep = 0.4pt

%\section*{SUPPLEMENTAL METHODS}
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Table S1 %
\renewcommand{\arraystretch}{1.2}

\begin{table}[h]
    \begin{center}
    \caption[]{List of maize landraces used in this study\hspace*{9.1cm}}  
{\fontsize{7}{10}\selectfont
%{\small
    \begin{tabular}{llllllllll}
        \hline\hline
       & & & \\[-4mm] 
	 ID$^a$	&	USDA ID	&	Population	&	Landrace	&	Locality	&	Latitude	&	Longitude	&	Elevation	&	Origin	\\[0.0cm]
	\hline 
	& & & \\[-4mm] 
{\bf RIMMA0409}	&	PI 478968	&	Mexico 	&	Tepecintle	&	Chiapas, Mexico	&	15.4 	&	-92.9 	&	107	&	USDA	\\
RIMMA0410	&	PI 478970	&	Lowland	&	Vandeno	&	Chiapas, Mexico	&	15.4 	&	-92.9 	&	107	&	USDA	\\
{\bf RIMMA0433}	&	PI 490825	&		&	Nal Tel ATB	&	Chiquimula, Guatemala	&	14.7 	&	-89.5 	&	457	&	USDA	\\
{\bf RIMMA0441}	&	PI 515538	&		&	Coscomatepec	&	Veracruz, Mexico	&	19.2 	&	-97.0 	&	1320	&	USDA	\\
{\bf RIMMA0615}	&	PI 628480	&		&	Tuxpeno	&	Puebla, Mexico	&	20.1 	&	-97.2 	&	152	&	USDA	\\
{\bf RIMMA0619}	&	PI 645772	&		&	Pepitilla	&	Guerrero, Mexico	&	18.4 	&	-99.5 	&	747	&	USDA	\\
{\bf RIMMA0628}	&	PI 646017	&		&	Tuxpeno Norteno	&	Tamaulipas, Mexico	&	23.3 	&	-99.0 	&	300	&	USDA	\\
{\bf RIMMA0696}	&	Ames 28568	&		&	Tuxpeno	&	El Progreso, Guatemala	&	16.5 	&	-90.2 	&	30	&	Goodman	\\
{\bf RIMMA0700}	&	NSL 291626	&		&	Olotillo	&	Chiapas, Mexico	&	16.8 	&	-93.2 	&	579	&	Goodman	\\
{\bf RIMMA0701}	&	PI 484808	&		&	Olotillo	&	Chiapas, Mexico	&	16.6 	&	-92.7 	&	686	&	Goodman	\\
{\bf RIMMA0702}	&	Ames 28534	&		&	Negro de Tierra Caliente	&	Sacatepequez, Guatemala	&	14.5 	&	-90.8 	&	1052	&	Goodman	\\
{\bf RIMMA0703}	&	NSL 283390	&		&	Nal Tel	&	Yucatan, Mexico	&	20.8 	&	-88.5 	&	30	&	Goodman	\\
{\bf RIMMA0709}	&	Ames 28452	&		&	Tehua	&	Chiapas, Mexico	&	16.5 	&	-92.5 	&	747	&	Goodman	\\
{\bf RIMMA0710}	&	PI 478988	&		&	Tepecintle	&	Chiapas, Mexico	&	15.3 	&	-92.6 	&	91	&	Goodman	\\
{\bf RIMMA0712}	&	NSL 291696 CYMT	&		&	Oloton	&	Baja Verapaz, Guatemala	&	15.3 	&	-90.3 	&	1220	&	Goodman	\\
{\bf RIMMA0716}	&	Ames 28459	&		&	Zapalote Grande	&	Chiapas, Mexico	&	15.3 	&	-92.7 	&	91	&	Goodman	\\
{\bf RIMMA0720}	&	PI 489372	&		&	Negro de Tierra Caliente	&	Guatemala	&	15.5 	&	-88.9 	&	39	&	Goodman	\\
{\bf RIMMA0721}	&	Ames 28485	&		&	Nal Tel ATB	&	Chiquimula, Guatemala	&	14.6 	&	-90.1 	&	915	&	Goodman	\\
{\bf RIMMA0722}	&	Ames 28564	&		&	Dzit Bacal	&	Jutiapa, Guatemala	&	14.3 	&	-89.7 	&	737	&	Goodman	\\
{\bf RIMMA0727}	&	Ames 28555	&		&	Comiteco	&	Guatemala	&	14.4 	&	-90.5 	&	1151	&	Goodman	\\
{\bf RIMMA0729}	&	PI 504090	&		&	Tepecintle	&	Guatemala	&	15.4 	&	-89.7 	&	122	&	Goodman	\\
{\bf RIMMA0730}	&	Ames 28517	&		&	Quicheno Late	&	Sacatepequez, Guatemala	&	14.5 	&	-90.8 	&	1067	&	Goodman	\\
{\bf RIMMA0731}	&	PI 484137	&		&	Bolita	&	Oaxaca, Mexico	&	16.8 	&	-96.7 	&	1520	&	Goodman	\\
{\bf RIMMA0733}	&	PI 479054	&		&	Zapalote Chico	&	Oaxaca, Mexico	&	16.6 	&	-94.6 	&	107	&	Goodman	\\
	\hline 
	& & & \\[-4mm] 
{\bf RIMMA0416}	&	PI 484428	&	Mexico	&	Cristalino de Chihuahua	&	Chihuahua, Mexico	&	29.4 	&	-107.8 	&	2140	&	NA	\\
{\bf RIMMA0417}	&	PI 484431	&	Highland	&	Azul	&	Chihuahua, Mexico	&	28.6 	&	-107.5 	&	2040	&	USDA	\\
{\bf RIMMA0418}	&	PI 484476	&		&	Gordo	&	Chihuahua, Mexico	&	28.6 	&	-107.5 	&	2040	&	USDA	\\
{\bf RIMMA0421}	&	PI 484595	&		&	Conico	&	Puebla, Mexico	&	19.9 	&	-98.0 	&	2250	&	USDA	\\
{\bf RIMMA0422}	&	PI 485071	&		&	Elotes Conicos	&	Puebla, Mexico	&	19.1 	&	-98.3 	&	2200	&	USDA	\\
{\bf RIMMA0423}	&	PI 485116	&		&	Cristalino de Chihuahua	&	Chihuahua, Mexico	&	29.2 	&	-108.1 	&	2095	&	NA	\\
{\bf RIMMA0424}	&	PI 485120	&		&	Apachito	&	Chihuahua, Mexico	&	28.0 	&	-107.6 	&	2400	&	USDA	\\
{\bf RIMMA0425}	&	PI 485128	&		&	Palomero Tipo Chihuahua	&	Chihuahua, Mexico	&	26.8 	&	-107.1 	&	2130	&	USDA	\\
{\bf RIMMA0614}	&	PI 628445	&		&	Mountain Yellow	&	Jalisco, Mexico	&	20.0 	&	-103.8 	&	2060	&	USDA	\\
{\bf RIMMA0616}	&	PI 629202	&		&	Zamorano Amarillo	&	Jalisco, Mexico	&	20.8 	&	-102.8 	&	1800	&	USDA	\\
{\bf RIMMA0620}	&	PI 645786	&		&	Celaya	&	Guanajuato, Mexico	&	20.2 	&	-100.9 	&	1799	&	USDA	\\
{\bf RIMMA0621}	&	PI 645804	&		&	Zamorano Amarillo	&	Guanajuato, Mexico	&	21.1 	&	-101.7 	&	1870	&	USDA	\\
{\bf RIMMA0623}	&	PI 645841	&		&	Palomero de Jalisco	&	Jalisco, Mexico	&	20.0 	&	-103.7 	&	2520	&	USDA	\\
{\bf RIMMA0625}	&	PI 645984	&		&	Cacahuacintle	&	Puebla, Mexico	&	19.0 	&	-97.4 	&	2600	&	USDA	\\
RIMMA0626	&	PI 645993	&		&	Arrocillo Amarillo	&	Puebla, Mexico	&	19.9 	&	-97.6 	&	2260	&	USDA	\\
{\bf RIMMA0630}	&	PI 646069	&		&	Arrocillo Amarillo	&	Veracruz, Mexico	&	19.8 	&	-97.3 	&	2220	&	USDA	\\
{\bf RIMMA0670}	&	Ames 28508	&		&	San Marceno	&	San Marcos, Guatemala	&	15.0 	&	-91.8 	&	2378	&	Goodman	\\
{\bf RIMMA0671}	&	Ames 28538	&		&	Salpor Tardio	&	Solola, Guatemala	&	14.8 	&	-91.3 	&	2477	&	Goodman	\\
{\bf RIMMA0672}	&	PI 483613	&		&	Chalqueno	&	Mexico, Mexico	&	19.7 	&	-99.1 	&	2256	&	Goodman	\\
{\bf RIMMA0674}	&	PI 483617	&		&	Toluca	&	Mexico, Mexico	&	19.3 	&	-99.7 	&	2652	&	Goodman	\\
{\bf RIMMA0677}	&	Ames 28476 	&		&	Conico Norteno	&	Zacatecas, Mexico	&	21.4 	&	-102.9 	&	1951	&	Goodman	\\
{\bf RIMMA0680}	&	Ames 28448	&		&	Tabloncillo	&	Jalisco, Mexico	&	20.4 	&	-102.2 	&	1890	&	Goodman	\\
{\bf RIMMA0682}	&	PI 484571	&		&	Tablilla de Ocho	&	Jalisco, Mexico	&	22.1 	&	-103.2 	&	1700	&	Goodman	\\
{\bf RIMMA0687}	&	Ames 28473	&		&	Conico Norteno	&	Queretaro, Mexico	&	20.4 	&	-100.0 	&	1921	&	Goodman	\\[-0.1mm]	
	                         %&                       &                                 & \emph{BoS-68}  & AB298905       &  AB054737\\[-0.1mm]	
	\hline\hline
\multicolumn{9}{l}{$^a$ GBS data are available for the accessions in bold font.}\\
    \end{tabular}}

\end{center} 

\end{table}

\clearpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\setcounter{table}{0}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Table S1 %
\renewcommand{\arraystretch}{1.2}

\begin{table}[h]
    \begin{center}
    \caption[]{(continued)\hspace*{13.5cm}}  
{\fontsize{7}{10}\selectfont
%{\small
    \begin{tabular}{llllllllll}
        \hline\hline
       & & & \\[-4mm] 
	 ID	&	USDA ID	&	Population	&	Landrace	&	Locality	&	Latitude	&	Longitude	&	Elevation (m)	&	Origin	\\[0.0cm]
	\hline 
	& & & \\[-4mm] 
{\bf RIMMA0388}	&	PI 443820	&	S. America	&	Amagaceno	&	Antioquia, Colombia	&	6.9 	&	-75.3 	&	1500	&	USDA	\\
{\bf RIMMA0389}	&	PI 444005	&	Lowland	&	Costeno	&	Atlantico, Colombia	&	10.4 	&	-74.9 	&	7	&	USDA	\\
{\bf RIMMA0390}	&	PI 444254	&		&	Comun	&	Caldas, Colombia	&	4.5 	&	-75.6 	&	353	&	USDA	\\
RIMMA0391	&	PI 444296	&		&	Andaqui	&	Caqueta, Colombia	&	1.4 	&	-75.8 	&	700	&	USDA	\\
{\bf RIMMA0392}	&	PI 444309	&		&	Andaqui	&	Caqueta, Colombia	&	1.8 	&	-75.6 	&	555	&	USDA	\\
{\bf RIMMA0393}	&	PI 444473	&		&	Costeno	&	Cordoba, Colombia	&	8.3 	&	-75.2 	&	100	&	USDA	\\
{\bf RIMMA0394}	&	PI 444621	&		&	Pira	&	Cundinamarca, Colombia	&	4.8 	&	-74.7 	&	1000	&	USDA	\\
{\bf RIMMA0395}	&	PI 444731	&		&	Negrito	&	Choco, Colombia	&	8.5 	&	-77.3 	&	30	&	USDA	\\
{\bf RIMMA0396}	&	PI 444834	&		&	Caqueteno	&	Huila, Colombia	&	2.6 	&	-75.3 	&	1100	&	USDA	\\
{\bf RIMMA0397}	&	PI 444897	&		&	Negrito	&	Magdalena, Colombia	&	11.6 	&	-72.9 	&	50	&	USDA	\\
{\bf RIMMA0398}	&	PI 444923	&		&	Puya	&	Magdalena, Colombia	&	9.4 	&	-75.7 	&	27	&	USDA	\\
{\bf RIMMA0399}	&	PI 444954	&		&	Cariaco	&	Magdalena, Colombia	&	10.2 	&	-74.1 	&	250	&	USDA	\\
{\bf RIMMA0403}	&	PI 445163	&		&	Pira Naranja	&	Narino, Colombia	&	1.3 	&	-77.5 	&	1000	&	USDA	\\
{\bf RIMMA0404}	&	PI 445322	&		&	Puya Grande	&	Norte de Santander, Colombia	&	7.3 	&	-72.5 	&	1500	&	USDA	\\
RIMMA0405	&	PI 445355	&		&	Puya	&	Norte de Santander, Colombia	&	8.4 	&	-73.3 	&	1100	&	USDA	\\
{\bf RIMMA0406}	&	PI 445514	&		&	Yucatan	&	Tolima, Colombia	&	5.0 	&	-74.9 	&	450	&	USDA	\\
RIMMA0407	&	PI 445528	&		&	Pira	&	Tolima, Colombia	&	4.2 	&	-74.9 	&	450	&	USDA	\\
{\bf RIMMA0428}	&	PI 485354	&		&	Aleman	&	Huanuco, Peru	&	-9.3 	&	-76.0 	&	700	&	NA	\\
{\bf RIMMA0462}	&	PI 445073	&		&	Amagaceno	&	Narino, Colombia	&	1.6 	&	-77.2 	&	1700	&	USDA	\\
{\bf RIMMA0690}	&	PI 444946	&		&	Puya	&	Magdalena, Colombia	&	8.3 	&	-73.6 	&	250	&	Goodman	\\
{\bf RIMMA0691}	&	PI 445391	&		&	Cacao	&	Santander, Colombia	&	6.6 	&	-73.1 	&	1098	&	NA	\\
{\bf RIMMA0707}	&	PI 487930	&		&	Tuxpeno	&	Ecuador	&	-1.1 	&	-80.5 	&	30	&	Goodman	\\
{\bf RIMMA0708}	&	PI 488376	&		&	Yunquillano F Andaqui	&	Ecuador	&	-3.5 	&	-78.6 	&	1098	&	Goodman	\\
	\hline 
	& & & \\[-4mm] 
{\bf RIMMA0426}	&	PI 485151	&	S. America	&	Rabo de Zorro	&	Ancash, Peru	&	-9.1 	&	-77.8 	&	2500	&	NA	\\
{\bf RIMMA0430}	&	PI 485362	&	Highland	&	Sarco	&	Ancash, Peru	&	-9.2 	&	-77.7 	&	2585	&	NA	\\
{\bf RIMMA0431}	&	PI 485363	&	 	&	Perlilla	&	Huanuco, Peru	&	-8.7 	&	-77.1 	&	2900	&	NA	\\
{\bf RIMMA0436}	&	PI 514723	&		&	Morocho Cajabambino	&	Amazonas, Peru	&	-6.2 	&	-77.9 	&	2200	&	NA	\\
{\bf RIMMA0437}	&	PI 514752	&		&	Ancashino	&	Ancash, Peru	&	-9.3 	&	-77.6 	&	2688	&	NA	\\
{\bf RIMMA0438}	&	PI 514809	&		&	Maranon	&	Ancash, Peru	&	-8.7 	&	-77.4 	&	2820	&	NA	\\
RIMMA0439	&	PI 514969	&		&	Maranon	&	La Libertad, Peru	&	-8.5 	&	-77.2 	&	2900	&	NA	\\
{\bf RIMMA0464}	&	PI 571438	&		&	Chullpi	&	Huancavelica, Peru	&	-12.3 	&	-74.7 	&	1800	&	USDA	\\
{\bf RIMMA0465}	&	PI 571457	&		&	Huarmaca	&	Piura, Peru	&	-5.6 	&	-79.5 	&	2300	&	USDA	\\
{\bf RIMMA0466}	&	PI 571577	&		&	Confite Puneno	&	Apurimac, Peru	&	-14.3 	&	-72.9 	&	3600	&	USDA	\\
{\bf RIMMA0467}	&	PI 571871	&		&	Paro	&	Apurimac, Peru	&	-13.6 	&	-72.9 	&	2800	&	USDA	\\
{\bf RIMMA0468}	&	PI 571960	&		&	Sarco	&	Ancash, Peru	&	-9.4 	&	-77.2 	&	3150	&	USDA	\\
{\bf RIMMA0473}	&	PI 445114	&		&	Sabanero	&	Narino, Colombia	&	1.1 	&	-77.6 	&	3104	&	USDA	\\
{\bf RIMMA0656}	&	Ames 28799	&		&	Culli	&	Jujuy, Argentina	&	-23.2 	&	-65.4 	&	2287	&	Goodman	\\
{\bf RIMMA0657}	&	NSL 286594	&		&	Chake Sara	&	Bolivia	&	-17.5 	&	-65.7 	&	2201	&	Goodman	\\
{\bf RIMMA0658}	&	NSL 286812	&		&	Uchuquilla	&	Bolivia	&	-21.8 	&	-64.1 	&	1948	&	Goodman	\\
{\bf RIMMA0661}	&	PI 488066	&		&	Chillo	&	Ecuador	&	-2.9 	&	-78.7 	&	2195	&	Goodman	\\
{\bf RIMMA0662}	&	NSL 287008	&		&	Cuzco	&	Ecuador	&	0.0 	&	-78.0 	&	2195	&	Goodman	\\
{\bf RIMMA0663}	&	PI 488102	&		&	Mishca	&	Ecuador	&	0.4 	&	-78.2 	&	2067	&	Goodman	\\
{\bf RIMMA0664}	&	PI 488113	&		&	Blanco Blandito	&	Ecuador	&	0.4 	&	-78.4 	&	2122	&	Goodman	\\
{\bf RIMMA0665}	&	PI 489324	&		&	Racimo de Uva	&	Ecuador	&	-0.9 	&	-78.9 	&	2931	&	Goodman	\\
{\bf RIMMA0667}	&	Ames 28737	&		&	Patillo	&	Chuquisaca, Bolivia	&	-21.8 	&	-64.1 	&	2201	&	NA	\\
RIMMA0668	&	Ames 28668	&		&	Granada	&	Puno, Peru	&	-14.9 	&	-70.6 	&	3925	&	Goodman	\\[-0.1mm] 
	                         %&                       &                                 & \emph{BoS-68}  & AB298905       &  AB054737\\[-0.1mm]	
	\hline\hline
	\multicolumn{9}{l}{$^a$ GBS data are available for the accessions in bold font.}\\
    \end{tabular}}
    \label{srkid}

\end{center} 

\end{table}

\clearpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%




%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\arraystretch}{1.1}
\begin{table}[h]

\begin{center}
 \caption[]{Inference of demographic parameters\hspace*{0.3cm}}
  \textbf{}\\[-2mm]
{\fontsize{8}{11}\sf
    \begin{tabular}{ccccccccl} \hline\hline
       & & \\[-3mm]
     Mexico  & \multicolumn{2}{c}{Model IA}  \\[0.1cm]
    \hline
    & & \\[-3mm]
   & Likelihood & $-$3052.34  \\
  &$N_B$ & 148,500 \\
  &$N_C$ & 148,500 \\   
  &$N_1$ & 62,370 \\ 
  &$N_2$ & 86,130 \\ 
  &$N_{2P}$   & 86,130    \\ 
      \hline
    & & \\[-3mm]
    S. America  & \multicolumn{2}{c}{Model IA}  \\[0.1cm]
        \hline
    & & \\[-3mm]
     & Likelihood &  $-$2717.64  \\
       &$N_B$ & 76,500 \\     	
       &$N_C$ & 76,500 \\
      &$N_1$ & 74,205           \\ 
      &$N_2$ & 2,295       \\      
      &$N_{2P}$   & 346,545   \\[1mm]
    \hline\hline
\multicolumn{3}{l}{The description of $\alpha, \beta$ and $\gamma$ is in Figure~3.}\\
\multicolumn{3}{l}{$\sigma$ is a relative size of $N_B$ to $N_C$ ($N_B=\sigma N_C$).}\\
    \end{tabular}
    \label{supp:param}  % caption is needed to make this work
}
\end{center}
\end{table}
\renewcommand{\arraystretch}{1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\arraystretch}{1.1}
\begin{table}[h]

\begin{center}
 \caption[]{Summary of PHS test\hspace*{6.1cm}}
  \textbf{}\\[-2mm]
{\fontsize{8}{11}\sf
    \begin{tabular}{lllllcccccl} \hline\hline
       & & \\[-3mm]
     Population  & Pattern of adaptation & No. SNPs & No. SNPs supported by PHS test \\[0.1cm]
    \hline
    & & \\[-3mm]
   Mexico & Highland adaptation & 264 & 172 (65.2\%)  \\
               & Lowland adaptation & 101 & 66 (65.3\%)  \\
   S. America & Highland adaptation & 164 & 230 (71.3\%)  \\
                     & Lowland adaptation & 70 & 50 (71.4\%)  \\[0.1cm]
    \hline\hline
    \end{tabular}
    \label{supp:phs}  % caption is needed to make this work
}
\end{center}
\end{table}
\renewcommand{\arraystretch}{1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\arraystretch}{1.1}
\begin{table}[h]

\begin{center}
 \caption[]{$F_{CT}$ between \emph{parviglumis} and \emph{mexicana}\hspace*{0.3cm}}
  \textbf{}\\[-2mm]
{\fontsize{7}{11}\sf
    \begin{tabular}{lcccccccl} 
    \hline\hline
       
Mexico     & \multicolumn{3}{c}{Number of SNPs}  \\
                                  & Significant & NS          & Proportion  \\
Significant $F_{CT}$ & 25              &   337       & 0.077\\ 
NS                             & 299            &  18,493   & 0.018\\
      \hline
    & & \\[-3mm]
S. America     & \multicolumn{3}{c}{Number of SNPs} \\
                                     & Significant & NS           & Proportion  \\
Significant $F_{CT}$    & 10              &   327        & 0.070\\ 
NS                                & 133            &  17,518    & 0.018\\[1mm]
    \hline\hline
  %\multicolumn{4}{l}{$^{a}$ \textcolor{red}{Maybe we should use the number of genetic unit.}}\\
  % \multicolumn{4}{l}{$^{a}$ \textcolor{red}{Sum of the number of SNPs != 91,779 because I excluded SNPs exhibiting}}\\[-0.5mm]
  % \multicolumn{4}{l}{ \textcolor{red}{     monomorphic in Mexico or in South America.}}\\
    \end{tabular}
    \label{tanja}  % caption is needed to make this work
}
\end{center}
\end{table}
\renewcommand{\arraystretch}{1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%=
\begin{table}[h]	
	\caption[]{$F_{ST}$ outlier SNPs and \emph{mexicana} introgression\hspace*{0.3cm}} 
			\centering
			\textbf{}\\[-2mm]
			{\fontsize{7}{11}\sf
		    \begin{tabular}{llcc} \hline\hline
		    & & \\[-3mm]
		    Introgression status & Population & $F_{ST}$ outlier SNPs & All other SNPs \\
		    \hline
		   Introgressed & Mexico & 114 & 1953 \\
		 	& S. America & 26 & 1721 \\
		   Not introgressed & Mexico & 558 & 73892 \\
		 	& S. America & 379 & 60666 \\
		      \hline\hline
		    \end{tabular}
		    \label{supp:introgressed}  % caption is needed to make this work
		}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\arraystretch}{1.1}
\begin{table}[h]

\begin{center}
 \caption[]{ms command \st{you don't use this table?}\hspace*{13.3cm}}
  \textbf{}\\[-2mm]
{\fontsize{7}{11}\sf
    \begin{tabular}{llcccccccl} \hline\hline
       & & \\[-3mm]
     Model I for Mexico  populations   \\
     Population 1: Mexico lowland population   \\
     Population 2: Mexico highland population   \\
  -I 2 $n_{m1}$ $n_{m2}$ -n 1 0.3496 -n 2 0.5704 -ej 0.01 2 1 -en 0.01 1 0.92 -en 0.0133 1 0.0163 -en 0.015 1 1.0 \\ 
      \hline
    & & \\[-3mm]
     Model II for Mexico  populations   \\
     Population 1: Mexico lowland population   \\
     Population 2: Mexico highland population   \\
     Population 3: \emph{mexicane} population   \\
-I 2 $n_{m1}$ $n_{m2}$ -n 1 1.14 -n 2 0.36 -es 0.01 2 0.8 -en 0.01 3 1.0667 -ej 0.01 2 1 -en 0.01 1 1.5 -en 0.0133 1 0.0163 -en 0.015 1 1.0 -ej 0.1 3 1 \\ 
      \hline
    & & \\[-3mm]
    Model I for SA  populations   \\
     Population 1: SA lowland population   \\
     Population 2: SA highland population   \\
  -I 2 $n_{s1}$ $n_{s2}$ -n 1 0.5044 -n 2 1.3728 -g 2 671.60 -ej 0.006667 2 1 -eg 0.006667 2 0.0 -en 0.00667 1 0.52 -en 0.01333 1 0.0163 -en 0.015 1 1.0 \\ 
      \hline
    & & \\[-3mm]
    Model III for SA  populations   \\
    Population 1: Mexico lowland population   \\
     Population 2: SA lowland population   \\
     Population 3: SA highland population   \\
  -I 3 $n_{m1}$ $n_{s1}$ $n_{s2}$ -n 1 0.64 -n 2 0.342 -n 3 0.972 -g 3 598.35 -ej 0.006667 3 2 -eg 0.006667 3 0.0 -en 0.006667 2 0.36 -ej 0.01 2 1\\
   -en 0.01 1 1 -en 0.0133 1 0.0163 -en 0.015 1 1.0 \\  [1mm]
    \hline\hline
\multicolumn{1}{l}{Sample size of Mexico lowland, Mexico highland, SA lowland and SA highland populations are denoted by $n_{m1}$, $n_{m2}$, $n_{s1}$ and $n_{s2}$, respectively.}\\
    \end{tabular}
    \label{commandline}  % caption is needed to make this work
}
\end{center}
\end{table}
\renewcommand{\arraystretch}{1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.5\columnwidth]{fig/bioclm.pdf}
    \caption{Annual mean temperature and annual precipitation in the sampling locations of the maize landraces across Americas.
    Red, blue, yellow and light blue bars represent Mexican lowland, Mexican highland, S. American lowland and S. American highland
    populations, respectively.  
    }
    \label{supp:colfreq}
  \end{center}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.4\columnwidth]{fig/Col.pdf}
    \caption{Correlation of allele frequencies between GBS (\emph{x}-axes) and MaizeSNP50 (\emph{y}-axes) data.  We used overlapped SNPs with $n\geq40$ for both data sets.  Correlation coefficient is 0.890 ($P<10^{-5}$ by permutation test with $10^5$ replications).}
    \label{supp:correl_freq}
  \end{center}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.4\columnwidth]{fig/kk.pdf}
    \caption{Likelihood of STRUCTURE analysis given $K$.  The \emph{x}-axes represents $K$ and the \emph{y}-axes represents likelihood.}
    \label{supp:struct}
  \end{center}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% FIGURE
\begin{figure}[h]   
  \begin{center}
   \vspace{-0mm}
   %\includegraphics[width=0.23\textwidth]{figs/model}
   \includegraphics[width=0.45\textwidth]{fig/Fig5}
   \renewcommand{\baselinestretch}{0.9}
   \vspace{-3mm}
   \caption{Observed and expected distributions of $F_{ST}$ values in GBS (A) and MaizeSNP50 data (B).  The \emph{x}-axes represent $F_{ST}$ values.  The \emph{y}-axes represent the frequency of SNPs with $F_{ST}$ values within a bin of 0.05 size.  Red dots and solid lines indicate observed and expected distributions. 
   }
\vspace{-6mm}
    \label{FstDist}
  \end{center}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% FIGURE

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.6\columnwidth]{fig/bot2.pdf}
    \caption{Q-Q plot for $-$log$_{10}$-scaled \emph{P}-values of population differentiation between lowland and highland populations. (A) Model IA \emph{v.s.} Model IB in Mexico, (B) Model IA \emph{v.s.} Model II in S. America, (C) Model with \emph{v.s.} without bottleneck in Mexico and (D) Model with \emph{v.s.} without bottleneck in S. America.}
    \label{fig:qq}
  \end{center}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h]
  \begin{center}
    \includegraphics[width=0.7\columnwidth]{fig/LD.pdf}
    \caption{Pattern of decay of linkage equilibrium in Mexico (A) and South America (B).  Red and blue dots represent low- and highland population, respectively.  $r^2$ values were calculated as a statistics and averaged within 10-bp bins of distance between SNPs.  The \emph{x}- and \emph{y}-axes represent distance between SNPs (kb) and average $r^2$ values.}
    \label{supp:LD}
  \end{center}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\newpage
\suppl
\section*{File S1}

\section*{Directionality of adaptation} \label{sec:supptextsho}
%\noindent {\Large \textit{Text~S1} }

We classified the patterns of allelic differentiation among highland and lowland populations in Mexico and S. America together with the information of \emph{parviglumis} in an \emph{ad hoc} manner; the allelic differentiation pattern is consistent with highland or lowland adaptation scenario.  
In Figure~\ref{AllelePat}, we illustrate the frequency of putative ancestral and derived alleles in the five populations, drawn by red and blue, respectively.

First, we focus on the SNPs with the signature of adaptation only in Mexican populations (Figure~\ref{AllelePat}A).  
The first and second rows shows the typical patterns of highland adaptation with \emph{parviglumis} data available.
We simply assume that the allele in higher frequency in \emph{parviglumis} is ancestral.
%\jri{we have decent Tripsacum data now, should we go back and re-assess this?}

Both rows show the consistent pattern to highland adaptation in Mexico because the frequency of the putative derived allele in Mexican highlands is highly differentiated from those in both \emph{parviglumis} and Mexican lowlands.
The patterns in S. America are different between the first and second rows.
However, we do not take the patterns in S. American populations into account because there is no adaptive signature in S. American.
On the other hand, we should consider the allelic pattern in S. America in the case of the third row; we cannot utilize the information of \emph{parviglumis}.
It is impossible to infer the ancestral allele, so we assume the pattern is consistent with highland adaptation if one allele is in higher frequency in Mexican lowlands and S. American populations and the others is in higher frequency in Mexican highlands.
We classified the SNPs into lowland adaptation in the same way (from fourth to sixth rows in Figure~\ref{AllelePat}A).

Next, we consider the SNPs with the signatures of adaptation in both Mexico and S. America (Figure~\ref{AllelePat}B).
The pattern in the first row is consistent with parallel highland adaptation, whereas the second row shows parallel lowland adaptation. 
We cannot infer lowland or highland adaptation without the outgroup, so we ignore such SNPs.
The pattern in the third row is the special case: the allele frequency is similar between Mexican lowlands and S. American highlands and similar between Mexican highlands and S. American lowlands.
This pattern could be explained by that the SNP is linked to a read adaptive SNP and recombination breaks down the linkage between them.

Finally, we tested whether PHS test supports highland and lowland adaptation scenario.
Consider the case of highland adaptation.
We assumed that the putative derived allele is adaptive in highlands and checked whether the haplotype length is longer in highlands than that in lowlands.
However, haplotype length cannot be compared directly because the derived allele frequency is different between highlands and lowlands.
Thus, we compared the \emph{P}-values of PHS test as a indicator of haplotype length given allele frequency (Pr($PHS_{xA}\leq PHS_{null|p}$ in Materials and Methods).
We just say that the PHS test is consistent if the \emph{P}-value in highlands is smaller than the \emph{P}-value in lowlands (haplotype length is longer as \emph{P}-value is smaller).
The result is summarized in Table~S3.

\renewcommand{\thefigure}{\Roman{figure}}
\setcounter{figure}{0}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[b]
  \begin{center}
    \includegraphics[width=0.9\columnwidth]{fig/AllelePat.pdf}
    \caption{Illustration of allele frequency changes in maize and \emph{parviglumis}.  Red and blue bars represent the allele frequency of ancestral and derived, adaptive alleles, respectively.  The allele frequencies in the five populations are shown: \emph{parviglumis}, Mexican lowlands and highlands, and S. America lowlands and highlands. NA in \emph{parviglumis} indicates that there is no SNP data in the site.}
    \label{AllelePat}
  \end{center}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%















