\documentclass[twocolumn,twoside,letterpaper]{article} 

\usepackage{color}

\usepackage{geneticsT2}
\usepackage{times}
\usepackage{hyperref}

\addtolength{\oddsidemargin}{-.2cm}
\addtolength{\evensidemargin}{-1.2cm}

\addtolength{\textwidth}{1.5cm}
\addtolength{\topmargin}{-2cm}
\addtolength{\textheight}{3.5cm}

\renewcommand{\textfraction}{0.01}
\renewcommand{\topfraction}{0.99}
\renewcommand{\bottomfraction}{0.65}
\renewcommand{\floatpagefraction}{0.90}
\renewcommand{\dbltopfraction}{0.95}
\renewcommand{\dblfloatpagefraction}{0.80}
\renewcommand{\sfdefault}{phv}

% plr added
\usepackage{amsmath}
\usepackage{amssymb}
\newcommand{\E}{\mathbb{E}}
\renewcommand{\P}{\mathbb{P}}
\newcommand{\var}{\mathop{\mbox{Var}}}
\newcommand{\mutrate}{\lambda_{\text{mut}}}
\newcommand{\migrate}{\lambda_{\text{mig}}}
\newcommand{\Tmut}{T_{\text{mut}}}
\newcommand{\Tmig}{T_{\text{mig}}}


\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{}

\fancyfoot[LE,RO]{{\sfbf \thepage}}
\renewcommand{\headrulewidth}{0pt}
\fancypagestyle{plain}{
	\fancyhf{}
}

%editing commands (please leave in for now)
\newcommand{\jri}[1]{\textcolor{blue}{ \emph{\scriptsize  #1}} }
\newcommand{\st}[1]{\textcolor{red}{#1}}
\newcommand{\comst}[1]{\textcolor{red}{ \em{\scriptsize  #1}} }
\definecolor{mattgreen} {rgb} {0,0.6,0}
\newcommand{\mbh}[1]{\textcolor{mattgreen}{ \em{\scriptsize  #1}} }
\definecolor{peterpurple}{rgb}{.6,0,.6}
\newcommand{\plr}[1]{\textcolor{peterpurple}{ \emph{\scriptsize (#1)}} }

\usepackage[normalem]{ulem}
\def\dt{\bgroup
 \markoverwith{\lower-0.2ex\hbox
 {\kern-.03em\vbox{\hrule width.2em\kern0.45ex\hrule}\kern-.03em}}%
 \ULon}
\MakeRobust\dt
\usepackage[normalem]{ulem}
\def\dt{\bgroup
 \markoverwith{\lower-0.2ex\hbox
 {\kern-.03em\vbox{\hrule width.2em\kern0.45ex\hrule}\kern-.03em}}%
 \ULon}
\MakeRobust\dt

%%%%

\title{Independent molecular basis of convergent highland adaptation in maize}
\author{
 \small\sfbf{Shohei Takuno$^{\ast ,}$, Peter Ralph$^{\dag, \ddag}$, Sofiane Mezmouk$^{\ast}$, Kelly Swarts$^{\S}$, Rob J. Elshire$^{\S}$, Jeffrey C. Glaubitz$^{\S}$,}\\
   \small\sfbf{Edward S. Buckler$^{\S, \ast\ast}$, Matthew B. Hufford$^{\ast, \dag\dag}$, and Jeffrey Ross-Ibarra$^{\ast,\ddag\ddag,}$}\thanks{
Corresponding author:  Department of Plant Sciences, University of California, Davis, California 95616, USA. 
    E-mail: \mbox{rossibarra@ucdavis.edu}}\\[0.3cm]
   \small\sf $^{\ast}$Department of Plant Sciences, University of California, Davis, California 95616, USA,\\
   \small\sf $^\dag$Department of Evolution and Ecology, University of California, Davis, California 95616, USA,\\
   \small\sf $^\ddag$Molecular and Computational Biology, University of Southern California,  Los Angeles,California 90089-0371, USA,\\
   \small\sf $^\S$Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853-2703, USA,\\
   \small\sf $^{\ast\ast}$United States Department of Agriculture Agricultural Research Service, Ithaca,
NY 14853,\\
   \small\sf $^{\dag\dag}$Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa 50011, USA,\\
   \small\sf $^{\ddag\ddag}$The Center for Population Biology and the Genome Center, University of California, Davis, California 95616 , USA,
   %\small\sf  Present address: Graduate University for Advanced Studies, Hayama, Kanagawa 240-0193, Japan\\
}

 
\date{Revised manuscript for \emph{Genetics}, \today}

\abstract{
Convergent evolution occurs when multiple species/subpopulations adapt to similar environments via similar phenotypes. 
We investigate here the molecular basis of convergent adaptation in maize to highland climates in Mexico and South America using genome-wide SNP data. 
Taking advantage of archaeological data on the arrival of maize to the highlands, we infer demographic models for both populations, identifying evidence of a strong bottleneck and rapid expansion in South America.  
We use these models to then identify loci showing an excess of differentiation as a means of identifying putative targets of natural selection, and compare our results to expectations from recently developed theory on parallel adaptation.  
Consistent with predictions across a wide array of parameter space, we see limited evidence for convergent evolution at the nucleotide level in spite of strong similarities in overall phenotypes.  
Instead, we show that selection appears to have predominantly acted on standing genetic variation, and that introgression from wild teosinte populations appears to have played a role in adaptation in Mexican maize.

\usepackage{natbib}
\bibpunct{(}{)}{;}{a}{}{,}

\usepackage{amsmath}

\usepackage{graphicx}

\begin{document}

\maketitle

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% INTRO
\input{intro}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% INTRO

\input{methods}

\input{results}

\section*{Conclusions} \jri{ WE NEED A CONCLUSION! }
We successfully inferred/revealed the molecular basis of independent adaptation to highland climates in the Mexican and S. American maize populations.
For this aim, we utilized the high throughput genotyping technologies: GBS and Illumina MaizeSNP50 BeadChip platform.
We inferred the demography, and detected the candidates of adaptive loci to highland climates by analyzing $\approx 90,000$ SNPs.
The majority of adaptive variants were derived from standing genetic variation in the maize with a large effective population size as theory predicted.
Introgression events from wild highland maize are also important source of highland adaptation.
Surprisingly, despite the fact that the environments of Mexican and S. American highlands are very similar, we found that parallel adaptation is very rare. 
Our newly developed theory supported the rarity of parallel adaptation in maize (???).
Thus, maize highland adaptation could be a good example of convergent adaptation in natural populations.

\st{silly sentences.}
However, it is known that the effective recombination rate of maize is very high (CITE, Maud, PNAS).
Linkage disequilibrium rapidly decays and reaches plateau around 100 bp (Figure~\ref{supp:LD}).
The density of SNPs was roughly 1 SNP per 20 kb, so we could miss some of the adaptive variants.
We believe that our conclusion holds even if more dense SNP dataset can be used, but the future genotyping technology may solve this problem.

%1. We successfully inferred demography and detected the candidates of adaptive loci to highland climates in Mexico and South America by utilizing GBS and 55-k chip.

%2. The main conclusion is parallel adaptation is rare in maize highland adaptation.


\begin{acknowledgments}
  We appreciate the helpful comments of P. Morrell and the members of the Ross-Ibarra lab and Coop lab.   This project was supported by Agriculture and Food Research Initiative Competitive Grant 2009-01864 from the USDAâ€™s National Institute of Food and Agriculture and funding from the National Science Foundation IOS-1238014.
\end{acknowledgments}

\bibliography{MZpara1.bib,MZpara2.bib,plr-hilo.bib}
\bibliographystyle{geneticsT2}

\suppl



%%%%%%%
% put supplemental figures and tables here:
% e.g.
% \begin{figure}
% ...


%%%%%
% appendices to follow
\input{demographic-model-appendix}
\input{HighLowSup1} 

\end{document}

