\section*{Introduction}
\noindent Repeated evolution occurs when multiple species or populations independently adapt to similar environmental challenges via mutations at the same loci \cite[]{Wood_2005_15881688,Arendt_2008_18022278,Elmer_2011_21459472}.
Evolutionary genetic analysis of a wide range of species has provided evidence for multiple pathways of repeated evolution. 
One such route occurs when identical mutations arise independently and fix via natural selection in multiple populations. In humans, for example, malaria resistance due to mutations from Glu to Val at the sixth codon of the $\beta$-globin gene has arisen independently on multiple unique haplotypes  \cite[]{Currat_2002_11741197,Kwiatkowski_2005_16001361}.  
Repeated evolution can also be achieved when different mutations arise within the same locus yet produce similar phenotypic effects.  
Grain fragrance in rice appears to have evolved along these lines, as populations across East Asia have similar fragrances resulting from at least eight distinct loss-of-function alleles in the  \emph{BADH2} gene \cite[]{Kovach_2009_19706531}.  
Finally, repeated evolution may arise from natural selection acting on standing genetic variation in an ancestral population.  
In the three-spined stickleback, natural selection has repeatedly acted to reduce armor plating in independent colonizations of freshwater environments.  
Adaptation in these populations occurred from standing variation at the \emph{Eda} locus in marine populations \cite[]{Colosimo_2005_15790847}.  
We still know relatively little, however, about how common it is for repeated phenotypic evolution to be driven by common genetic changes or the relative frequencies of these different routes of repeated evolution.
\jri{another recent nice example, repeated gene loss in cyanogenic clovers \url{http://rstb.royalsocietypublishing.org/content/369/1648/20130347} or convergene evolution of electric shocks \url{http://www.sciencemag.org/content/344/6191/1522}; also feel like we need more intro here.  this mentions a few ideas, but not some of the interesting questions or relevant points about genetic architecture, target size, the role of migration, etc.}


Domesticated maize (\emph{Zea mays} ssp. \emph{mays}) provides an excellent opportunity to investigate the molecular basis of repeated evolution.  
Maize was domesticated from the wild teosinte \emph{Zea mays} ssp. \emph{parviglumis} (hereafter \emph{parviglumis}) in the lowlands of southwest Mexico $\sim$9,000 years before present (BP) \cite[]{Matsuoka_2002_11983901,Piperno_2009_19307570,vanHeerwaarden_2011_21189301}. 
After domestication, maize spread rapidly across the Americas, reaching the lowlands of South America and the high altitudes of the Mexican Central Plateau by $\sim$6,000 BP \cite[]{Piperno_2006_69}, and the Andean highlands $\sim$2,000 years later \cite[]{Perry_2006_16511492,Grobman_2012_22307642}. 
The transition from lowland to highland habitats spanned similar environmental gradients in Mexico and South America (Figure~\ref{supp:colfreq}) and presented a host of novel challenges that often accompany highland adaptation including reduced temperature, increased ultraviolet radiation, and reduced partial pressure of atmospheric gases \cite[]{Korner_2007_17988759}.
\jri{Sho: please check for hits in any UV related genes.  see \url{http://www.biomedcentral.com/1471-2229/12/92}, \url{http://www.reeis.usda.gov/web/crisprojectpages/0195696-maize-responses-to-uv-b-a-genomics-assessment.html}, and \url{http://onlinelibrary.wiley.com/doi/10.1111/j.1365-3040.2005.01329.x/full} as well as \cite[]{Casati2005}}

Common garden experiments in Mexico reveal that highland maize has successfully adapted to highland conditions \cite[]{Mercer2008}, and phenotypic comparisons between Mexican and South American populations are suggestive of repeated evolution.  
Landraces from both populations share a number of phenotypes not found in lowland populations, including dense macrohairs \cite[]{Wilkes_1977,Wellhausen1957:book}, stem pigmentation \cite[]{Wilkes_1977,Wellhausen1957:book}, and biochemical response to UV radiation \cite[]{Casati2005}. 
Genetic analyses of maize landraces from across the Americas indicate that the two highland populations are independently derived from their respective lowland populations \cite[]{Vigouroux_2008_21632329, vanHeerwaarden_2011_21189301}, so observed patterns of phenotypic similarity are not simply due to recent shared ancestry. 

Although there are no wild relatives of maize in South America, the teosinte \emph{Zea mays} ssp. \emph{mexicana} (hereafter \emph{mexicana}) is native to the highlands of central Mexico, where it is thought to have occurred since at least the last glacial maximum \cite[]{Ross-Ibarra_2009_19153259, Hufford_niche}. Phenotypic differences between \emph{mexicana} and \emph{parviglumis} mirror those between highland and lowland maize \cite[]{Lauter_2004_15342532} and population genetic analyses of the two subspecies reveal evidence of natural selection associated with altitudinal differences between \emph{mexicana} and \emph{parviglumis} \cite[]{Pyhajarvi2013}.  Landraces in the highlands of Mexico are often found in sympatry with  \emph{mexicana} and gene flow from \emph{mexicana} likely contributed to maize adaptation to the highlands \cite[]{Profford_2013}.

In this paper we set out to address a number of questions regarding highland adaptation in maize: 
What is the genetic architecture of highland adaptation? 
Do maize populations in the highlands of Mexico and South America show evidence of repeated evolution at the molecular level? 
How do observed patterns of repeated evolution compare to theoretical expectations?
We make use of SNP genotyping to characterize patterns of natural selection in highland maize and compare our results to expectations from theoretical models of repeated evolution.  
We estimate unique demographic histories in the highlands of Mexico and South America, and find evidence supporting our theoretical predictions that adaptation should be largely independent. 
Our population genetic analysis also supports an adaptive role for gene flow from \emph{mexicana} and highlights the contribution of standing variation to adaptation in both populations.