%For discussion:

While the large number of environmental factors implies that a large number of adaptive changes may be required to colonize     high altitude environments, evidence of selection in multiple mammal species consistent with the idea that parallel adaptation dep    ends on the genetic architecture of individual traits.\cite[]{Yi_2010_20595611,Simonson_2010_20466884,Storz_2007_17397259,Qiu_2012_    22751099}.

In summary, the progenitor populations of highland maize (lowland maize and \emph{parviglumis}) contain both genotypes of highly differentiated SNPs implying adaptation from standing variation.
However, standing variation may be difficult to distinguish from gene flow between lowland and highland populations due to their recent divergence in units of $N_A$.
An additional caveat is that our inference of adaptation from standing variation assumes that our highly differentiated SNPs are causal and not merely linked to the targets of adaptation.  A causal variant at some distance from the assayed SNP may have had the opportunity to recombine onto multiple backgrounds. 
Given the previously documented rapid decay of linkage disequilibrium in maize \cite[]{Tenaillon_2001_11470895,Remington_2001_11562485} that is also apparent in our data (supp fig.~X) we are also likely missing a number of selected loci not in LD with the SNPs assayed here.

%\st{(Or we can say without gene flow, we can well explain the observed pattern of polymorphisms).
%Lots of shared polymorphisms between maize and teosinte may support the standing variation hypo? Standing variation hypo holds as long as the effect of gene flow between maize and teosinte is negligible. 
%In fact, \cite{vanHeerwaarden_2011_21189301} estimated admixture between Mexico lowlands maize and \emph{parviglumis} to be very low. 
%Admixture from  \emph{parviglumis} to maize may alter the shape of the null distributions of $F_{ST}$ values.
%We checked even high admixture (up to 20\%) did not change the distributions (\cite{vanHeerwaarden_2011_21189301} estimated a few percentage of admixture).
%Thus, our $F_{ST}$ outlier approach is conservative on gene flow.}


%\subsection*{Others}
%Or independently selected genes in Mexico and South America may belong to the same metabolic? chemical? pathway.  
%It may be worth doing maizecys. \jri{ let's make sure we have figured out standing var for SNP frequencies and GU before doing maizecyc.  but indeed might be worth doing.  let's hold off for now, though.} \comst{yeah}


%Effective population size low $<$ high in Mexico and low $>$ high in South America



%Would be good to have a figure showing observed and expected overlap in SNPs and Genes.



%We obtained teosinte (both  and \emph{mexicana}) SNPs in 46 of 56 SNPs from .  39 SNPs exhibited polymorphisms in teosinte, indicating selective sweep from standing genetic variation.  The other seven SNPs showed monomorphic in teosinte.  Only in two of seven SNPs, the frequencies of derived alleles were increased in highland populations. 

%Note that a SNP with significant \emph{P} value is not necessarily the causative variant because we cannot rule out the possibility that the SNP is just linked to the true causative one.  

%, or the two SNPs are linked to the causative SNPs and recombination .

%    The distance of genetic units would be fairly long, compared to the rapid decay of linkage disequilibrium in maize (roughly 100 bp; Supp Fig. X) \cite[]{Tenaillon_2001_11470895}.  We just assumed that SNPs within a genetic unit have the same functional effect or link to the functional SNP, and based on this assumption, we screened SNPs showing Pattern C of parallel adaptation in Fig.~\ref{fig1}.  \textcolor{red}{Do you have a better idea?}  \jri{i like this but i think we need to be more explicit about relating our analyses back to the patterns.  are we doing this because of linkage or because of biology or both?} \mbh{It might be a good idea to mention linkage to the heuristic set up in Figure 1} 

% \jri{ so this is worth thinking about.  what do we call it if the same SNP was selected but in alternate directions?  i think you need to explain "recombination" further.  i assume you mean that both snp alleles occurred on some background that was then selected in parallel?  since we don't know what SNP was selected how do we categorize this?  Also, somewhere we need to make clear that our different models of parallel adaptation assume that if we see high $F_{ST}$ in two pops that it's because that snp is selected for.  It's just as possible to have the same SNP allele selected in both pops but because it was on different backgrounds! }

% \textcolor{red}{Parallel adaptation in lowland populations also occurred!!} \jri{ not sure how we know this was parallel adaptation in lowland... could be ancestral snp on a derived background (recombination), could be the ancestral allele is suddenly beneficial in a highland habitat (parallel adaptation in highland) or that the new derived allele is beneficial in lowland (parallel adaptation in lowland).  can we distinguish among these (or other) options? }

%Note that the result was not changed when using working gene set. 
%\jri{need to add working gene set not changing result into text.}

%A lot of adaptive SNPs are from standing variation.  That was confirmed by SNPs of teosinte in Hapmap v2. 

%\jri{is standing variation defined in teosinte or should it be defined in lowland ancestral pop? some SNPs that are monomorphic in teo might have been polymorphic in lowland ancestor? } \mbh {It seems if we're talking about highland adaptation from standing variation, it has to be from variation in lowland maize}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\arraystretch}{1.1}
\begin{table}[tb]

\begin{center}
 \caption[]{Parallel adaptation\hspace*{0.3cm}}
  \textbf{}\\[-2mm]
{\fontsize{7}{11}\sf
    \begin{tabular}{lllcccccl} \hline
       & & \\[-3mm]
     Description  & Number of genetic units\\[0.1cm]
    \hline
Pattern A or B     & 45 \\
Pattern C from standing variation & 16\\ 
Pattern C from standing variation and & 2\\
\ \ vi new mutations & \\
Total & 63\\[1mm]
    \hline
  \multicolumn{2}{l}{$^{a}$ \textcolor{red}{In this table, I did not use the information of teosinte polymorphisms.}}\\
    \end{tabular}
    \label{paraGU}  % caption is needed to make this work
}
\end{center}
\end{table}
\renewcommand{\arraystretch}{1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% pre
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\arraystretch}{1.1}
\begin{table}[tb]

\begin{center}
 \caption[]{Parallel adaptation\hspace*{0.3cm}}
  \textbf{}\\[-2mm]
{\fontsize{7}{11}\sf
    \begin{tabular}{lllcccccl} \hline
       & & \\[-3mm]
     Description  & Number of genetic units\\[0.1cm]
    \hline
Parallel adaptation     &  \\
Pattern A or B     & 45 \\
Pattern C from standing variation & 16\\ 
Pattern C from standing variation and & 2\\
\ \ vi new mutations & \\
Total & 63\\
      \hline
    & & \\[-3mm]
Only Mexico     &  \\
From standing variation & 635 \\
New mutations & 65 \\
Both mechanisms & 16 \\
Total & 716\\
      \hline
    & & \\[-3mm]
Only South America     &  \\
From standing variation & 458 \\
New mutations & 36 \\
Both mechanisms & 4 \\
Total & 498\\[1mm]
    \hline
  \multicolumn{2}{l}{$^{a}$ \textcolor{red}{Check teosinte polymorphisms on Hapmap2 later.}}\\
    \end{tabular}
    \label{paraGU}  % caption is needed to make this work
}
\end{center}
\end{table}
\renewcommand{\arraystretch}{1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


