\documentclass[onecolumn,oneside,letterpaper]{article} 

\usepackage{response}
%\usepackage{geneticsT2}

\usepackage{times}
\usepackage{color}

% rayout %
\addtolength{\oddsidemargin}{-2.75cm}
\addtolength{\evensidemargin}{-0.75cm}

\addtolength{\textwidth}{5.5cm}
\addtolength{\topmargin}{-3cm}
\addtolength{\textheight}{4.5cm}

%\parindent=4em
%\setlength{\parskip}{1ex plus 0.5ex minus 0.2ex} 

\setlength{\parindent}{0pt}
\setlength{\parskip}{10.0pt}
%\renewcommand{\itemindent}{50pt}
\setlength{\leftskip}{0.5cm}
\renewcommand{\labelitemi}{$-$}

\newcommand{\jri}[1]{\textcolor{blue}{ \emph{\scriptsize  #1}} }
\newcommand{\jristop}{\textcolor{red}{\hline \bf JRI STOPPED HERE\\ \hline}}
\newcommand{\st}[1]{\textcolor{red}{ #1}}
\newcommand{\comst}[1]{\textcolor{red}{ \em{\scriptsize  (#1)}} }
\definecolor{mattgreen} {rgb} {0,0.6,0}
\newcommand{\mbh}[1]{\textcolor{mattgreen}{ \em{\scriptsize  #1}} }
\definecolor{peterpurple}{rgb}{.6,0,.6}
\newcommand{\plr}[1]{\textcolor{peterpurple}{ \emph{\scriptsize (#1)}} }

\renewcommand{\textfraction}{0.01}
\renewcommand{\topfraction}{0.99}
\renewcommand{\bottomfraction}{0.65}
\renewcommand{\floatpagefraction}{0.90}
\renewcommand{\dbltopfraction}{0.95}
\renewcommand{\dblfloatpagefraction}{0.80}
\renewcommand{\sfdefault}{phv}

\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{}
\fancyfoot[CE,CO]{\thepage}
\renewcommand{\headrulewidth}{0pt}
\fancypagestyle{plain}{
	\fancyhf{}
}

% space of double hline in Table
\doublerulesep = 0.4pt

\title{Parallel adaptation to highland climate in maize domesticated populations}

\author{
 Shohei Takuno, Kelly Swarts, Matthew B. Hufford, Rob J. Elshire, Jeffrey C. Glaubitz, Edward S. Buckler and Jeffrey Ross-Ibarra
   }

\usepackage{natbib}
\bibpunct{(}{)}{;}{a}{}{,}

\usepackage{amsmath}

\usepackage{graphicx}

\begin{document}

\maketitle
Dear Editor - 

Thank you for soliciting these reviews, which were very helpful.   We believe we have responded to all of the reviewers' comments, hopefully to their satisfaction.  We also hope that the manuscript is improved as a result of the revisions.  

We provide detailed responses to the reviewers' comments below; the responses are in \textit{italics} and also bulleted for ease of identification.  
Briefly, the revisions are as follows:  
\st{Reviewer 1 made three non-compulsory suggestions, and we included analyses based on two of the three suggestions.  
These additional represent a good deal of extra work, but we believe they strengthen the paper. 
Reviewer 2 commented that the methods were unclear.  
In a couple of places I was uncertain as to what, exactly, this reviewer required, but we did substantially revise the description of methods by: i) including them in the narrative of the Results section and ii) expanding the methods section.   
Because the latter put the manuscript far over the page limit, some of the methods had to be moved to the SI Appendix.}  

I hope that these revisions will make the paper suitable for publication in Genetics. 

Thank you again! 

Sincerely, 

Jeffrey Ross-Ibarra

--------------------------------------------------------------------


AE comments: 

The ``Demographic model" is too detailed.  These fine-scaled demographic factors influence just two parameters: the effective population density and the rate of local diffusion.  Given all the uncertainties, it may be best just to consider a wide range of more or less plausible values. 
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Peter will reply this.  Thanks!!  hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

It is interesting to consider the relative plausibility that alleles that are favoured at high altitude either arose by mutation at high altitude, or moved from low altitudes where they are disfavoured.  However, the actual model is not spell out.  Are these habitats disjunct?  Or is there a continuous range? If the latter, does the selection coefficient change abruptly, or gradually?  It seems that there is a gap of distance R between the two habitats, but then, how does the maize get across?  This all seems obscure. 
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

More seriously, simple diffusion is not plausible for very long-term movement: assuming diffusion alone makes the chances of movement extremely sensitive to the parameters, as is apparent from the examples given.  Ultimately, what matters is the number of new mutations in the highland population, relative to the numbers of mutations that arise in the lowland, persist despite counter-selection, and move, one way or another, to the highlands.  It would be simpler just to say that rather than make over-elaborate diffusion models. 
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

In any case, it seems more plausible that adaptation is from standing variation, but picks up different allelic combinations in different high-altitude populations.  That is not at all unlikely, either by chance or because of epistasis which allows the same phenotype to come from different (and incompatible) gene combinations.  Actually, crossing experiments between high-altitude populations would be most informative here. 
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

The fixation probability result (attributed to Jagers 1975) is wrong.  Fixation probability is 2s(Ne/N).  For diploid sexualise, Ne=N if the offspring distribution is Poisson with mean 2, in which case the variance of offspring number is 2.  The error may have come from confusion between numbers of genes and numbers of offspring from a sexual pair.
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}


Reviewer 1 Reviewer's Comments to Author

I have reviewed the work of Takuno and colleagues on the independent genetic basis of adaptation in maize. The manuscript is well written, the questions that are addressed are well researched and are analyzed in a very thorough and complete fashion. I honestly don't have much to say about it, besides congratulating the authors on producing an excellent and thorough investigation of the demographics and genetic basic of adaptation in maize. The demonstration of the independent origin of highland maize is convincing and the role of adaptive introgression from Zea mays mexicana quite fascinating actually.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ We are grateful that the reviewer likes our paper.  The comments are very helpful!! }
\end{itemize}
\setlength{\parskip}{10.0pt}

Although I read the section on the theoretical evaluation of convergent evolution, followed its logical argument and agreed with it, I did not review it in details.  It is beyond my range of expertise.  I do wonder however how realistic these models are for domesticated species, where census sizes have probably little do to with effective population sizes, and where alleles could potentially be transported several hundreds (or more?) km away in a single generation by humans.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Peter and Jeff will handle it. }
\end{itemize}
\setlength{\parskip}{10.0pt}

I thought the use of the SNP array + GBS data to be slightly confusing, especially the high heterozygote error rate in the GBS data? However, the authors are aware of this and have treated the data accordingly.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ As we showed in Figure S2, despite the very high error rate of calling hererozygote SNPs in GBS, we found that estimated allele frequencies from both GBS and MaizeSNP50 are highly correlated.  Also, we carefully filtered out SNPs that showed the departure from Hardy-Weinberg equilibrium (HWE).  We tested different cut-offs of HWE, and obtained the similar results and conclusion.  Those were described in the left column in the page 3.  \comst{Did we clarify it in greater details in the text?} }
\end{itemize}
\setlength{\parskip}{10.0pt}  

Structure analyses:  Using only 3 replicate runs is relatively low, but the results should be robust to this and so I am not too worried.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Actually, we ran STRUCTURE more than 3 times for preliminary surveys, and confirmed that our final result is robust.  Thanks for the comment, but we keep the text unedited. }
\end{itemize}
\setlength{\parskip}{10.0pt}

The authors do not find much evidence of the same SNPs and genes underlying adaptation in highland south and meso-America. For personal interest, I would be interested to know if by comparing large segments of the genome (megabases scale), patterns would emerge.  This could imply that certain regions of the genome are more prone to be targets of selection  (because of factors such as variations in recombination or mutation rate). This question would be interesting, but not necessary for the current paper.    
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ We agreed, and have actually tried to test convergent adaptation by sliding window.  However, the density of our SNPs was not enough high to do this (1 SNP/10 or 20 kb).  Also, the very high recombination rate in the maize genome would limit the effect of selection to a narrow local genomic region (see also the rapid decay of LD in Figure S6), and therefore we thought this approach would not suitable in maize. }
\end{itemize}
\setlength{\parskip}{10.0pt}

This is a minor detail, but you start the abstract by defining convergent evolution, then jump right away to talking about convergent ``adaptation".  Perhaps stick to one or another? 
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ We agreed and thank this comment.  \st{I modified Abstract, Jeff.  Please check!!} }
\end{itemize}
\setlength{\parskip}{10.0pt}

Reviewer 2 Reviewer's Comments to Author

Tajuno and colleagues use highland and lowland maize populations from sites in Mesoamerica and South America to search for loci involved in adaptation to high altitude.  They identify models to describe demographic history and then use these as the basis for defining putatively adaptive variants based on highland/lowland differentiation.  Somewhat surprisingly, although they find that highly differentiated SNPs tend to come from the standing variation, few of the most differentiated variants are shared among high altitude sites. Instead, the majority of the variants are identified in only one contrasting population set.   
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Hello, my name is Takuno, but not Tajuno... }
\end{itemize}
\setlength{\parskip}{10.0pt}

The authors put quite a lot of effort into modeling demographic histories and are thus able to use a model-based approach to identify candidate adaptive variants.  This is an improvement over a simple tail-based outlier approach.  However, since the main message of the paper is that convergent rather than parallel evolution drove adaptive response, it is important to ensure that this finding is robust and not simply due to false positives or model mis- specification.  In general, in studies of this sort, it is difficult to assess the relative importance of parallel and convergent adaptive shifts. This is because many of the signals will be due to false positives, and these are reduced in the set of variants that share signatures of divergence between the two altitude contrasts.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ If false positive is a matter, we use more stringent cut-off.   We reached the same conclusion with more or
 less stringent cut-offs by model-based and model-free approaches.  }
\end{itemize}
\setlength{\parskip}{10.0pt}

Information about variant/gene function could help to strengthen the argument -- or to ensure that the argument is the correct one.  There is really no attempt to provide biological context for the results, but this could complement the findings and could potentially help to make them more convincing.  For example, are the loci identified as adaptive enriched for candidate genes or pathways involved in altitude adaptation? How do the set of loci showing parallel versus convergent adaptive signals compare functionally?   
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Sho will try this, but the annotation of maize is not as good as Arabidopsis... }
\end{itemize}
\setlength{\parskip}{10.0pt}

Additional/specific comments:  

To bolster the argument overall, it would be helpful to know whether there is any evidence of convergent adaptation to high altitude at the phenotypic level, for example from transplant experiments or from comparative phenotyping assays in a common garden or other shared environment.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

There are some very extreme outliers in the differentiation analysis (Figure 4). Do these make biological sense? Also, are relevant pathways/biological processes enriched in the less extreme tail?  Is there greater enrichment among the loci that are shared in the two high altitude populations? I suspect the number of false positive results among those differentiated in both high altitude populations would be lower than those that are differentiated in only one.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Sho will try this. }
\end{itemize}
\setlength{\parskip}{10.0pt}

There appears to be strong enrichment of variants with extreme PHS among the variants identified as adaptive. But as far as I could tell, no formal test was conducted. Moreover, I could not find the description of how the authors determined that these had signals from the PHS test. They write that they use the ?empirical quantile? but do not say which quantile is used to define significance.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Sho will edit this. }
\end{itemize}
\setlength{\parskip}{10.0pt}

Please explain more about the model for fitting genetic distances.  How was it decided that a 10th degree polynomial curve was the best fit for the genetic distances, and how was this model used in the analysis? (Page 5)  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

It was really interesting that the authors did not find evidence for a domestication bottleneck.  Since this is such a surprising result and one that is in stark contrast to the expectation, it would be helpful to see more about the robustness of this and to have a more complete comparison to previous work.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

Without some sort of evidence of functional significance, the result that different standing variants increase in frequency with altitude in different populations is not completely satisfying.  Since the variants that increase in frequency are as standing variation in both lowland populations, it seems surprising that the sets of putative adaptive variants are almost completely disjoint.  Without additional evidence for enrichment of the same or similar biological pathways or some other evidence that the outliers are involved in adaptation, this pattern seems most consistent with a finding of different sets of false positives in the two data sets.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ hoge }
\end{itemize}
\setlength{\parskip}{10.0pt}

?File S1: Directionality of adaptation? contains several typos: 
shows <- show 
American <- America 
others <- other 
I am not sure what the following sentence means: ?This pattern could be explained by that the SNP is linked to a read adaptive SNP and recombination breaks down the linkage between them.?   
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Sho will fix them. }
\end{itemize}
\setlength{\parskip}{10.0pt}

Figure I: The legend is insufficient to understand what is plotted here. Mainly, in panel A, I cannot find any description of what is represented in each row. Panel B is much more intuitive, but I am still not sure what ?parallel adaptation with recombination? means.  
\setlength{\parskip}{-5.0pt}
\begin{itemize}
\item \textit{ Sho will edit these. }
\end{itemize}
\setlength{\parskip}{10.0pt}

%\clearpage



\end{document}










